/*
Navicat MySQL Data Transfer

Source Server         : localhost_3306
Source Server Version : 50714
Source Host           : localhost:3306
Source Database       : cirrna

Target Server Type    : MYSQL
Target Server Version : 50714
File Encoding         : 65001

Date: 2018-03-10 13:06:22
*/

SET FOREIGN_KEY_CHECKS=0;

-- ----------------------------
-- Table structure for auth_group
-- ----------------------------
DROP TABLE IF EXISTS `auth_group`;
CREATE TABLE `auth_group` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `name` varchar(80) NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `name` (`name`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8;

-- ----------------------------
-- Records of auth_group
-- ----------------------------

-- ----------------------------
-- Table structure for auth_group_permissions
-- ----------------------------
DROP TABLE IF EXISTS `auth_group_permissions`;
CREATE TABLE `auth_group_permissions` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `group_id` int(11) NOT NULL,
  `permission_id` int(11) NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `auth_group_permissions_group_id_permission_id_0cd325b0_uniq` (`group_id`,`permission_id`),
  KEY `auth_group_permissio_permission_id_84c5c92e_fk_auth_perm` (`permission_id`),
  CONSTRAINT `auth_group_permissio_permission_id_84c5c92e_fk_auth_perm` FOREIGN KEY (`permission_id`) REFERENCES `auth_permission` (`id`),
  CONSTRAINT `auth_group_permissions_group_id_b120cbf9_fk_auth_group_id` FOREIGN KEY (`group_id`) REFERENCES `auth_group` (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8;

-- ----------------------------
-- Records of auth_group_permissions
-- ----------------------------

-- ----------------------------
-- Table structure for auth_permission
-- ----------------------------
DROP TABLE IF EXISTS `auth_permission`;
CREATE TABLE `auth_permission` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `name` varchar(255) NOT NULL,
  `content_type_id` int(11) NOT NULL,
  `codename` varchar(100) NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `auth_permission_content_type_id_codename_01ab375a_uniq` (`content_type_id`,`codename`),
  CONSTRAINT `auth_permission_content_type_id_2f476e4b_fk_django_co` FOREIGN KEY (`content_type_id`) REFERENCES `django_content_type` (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=28 DEFAULT CHARSET=utf8;

-- ----------------------------
-- Records of auth_permission
-- ----------------------------
INSERT INTO `auth_permission` VALUES ('1', 'Can add disease', '1', 'add_disease');
INSERT INTO `auth_permission` VALUES ('2', 'Can change disease', '1', 'change_disease');
INSERT INTO `auth_permission` VALUES ('3', 'Can delete disease', '1', 'delete_disease');
INSERT INTO `auth_permission` VALUES ('4', 'Can add cirrna', '2', 'add_cirrna');
INSERT INTO `auth_permission` VALUES ('5', 'Can change cirrna', '2', 'change_cirrna');
INSERT INTO `auth_permission` VALUES ('6', 'Can delete cirrna', '2', 'delete_cirrna');
INSERT INTO `auth_permission` VALUES ('7', 'Can add relationship', '3', 'add_relationship');
INSERT INTO `auth_permission` VALUES ('8', 'Can change relationship', '3', 'change_relationship');
INSERT INTO `auth_permission` VALUES ('9', 'Can delete relationship', '3', 'delete_relationship');
INSERT INTO `auth_permission` VALUES ('10', 'Can add log entry', '4', 'add_logentry');
INSERT INTO `auth_permission` VALUES ('11', 'Can change log entry', '4', 'change_logentry');
INSERT INTO `auth_permission` VALUES ('12', 'Can delete log entry', '4', 'delete_logentry');
INSERT INTO `auth_permission` VALUES ('13', 'Can add permission', '5', 'add_permission');
INSERT INTO `auth_permission` VALUES ('14', 'Can change permission', '5', 'change_permission');
INSERT INTO `auth_permission` VALUES ('15', 'Can delete permission', '5', 'delete_permission');
INSERT INTO `auth_permission` VALUES ('16', 'Can add group', '6', 'add_group');
INSERT INTO `auth_permission` VALUES ('17', 'Can change group', '6', 'change_group');
INSERT INTO `auth_permission` VALUES ('18', 'Can delete group', '6', 'delete_group');
INSERT INTO `auth_permission` VALUES ('19', 'Can add user', '7', 'add_user');
INSERT INTO `auth_permission` VALUES ('20', 'Can change user', '7', 'change_user');
INSERT INTO `auth_permission` VALUES ('21', 'Can delete user', '7', 'delete_user');
INSERT INTO `auth_permission` VALUES ('22', 'Can add content type', '8', 'add_contenttype');
INSERT INTO `auth_permission` VALUES ('23', 'Can change content type', '8', 'change_contenttype');
INSERT INTO `auth_permission` VALUES ('24', 'Can delete content type', '8', 'delete_contenttype');
INSERT INTO `auth_permission` VALUES ('25', 'Can add session', '9', 'add_session');
INSERT INTO `auth_permission` VALUES ('26', 'Can change session', '9', 'change_session');
INSERT INTO `auth_permission` VALUES ('27', 'Can delete session', '9', 'delete_session');

-- ----------------------------
-- Table structure for auth_user
-- ----------------------------
DROP TABLE IF EXISTS `auth_user`;
CREATE TABLE `auth_user` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `password` varchar(128) NOT NULL,
  `last_login` datetime(6) DEFAULT NULL,
  `is_superuser` tinyint(1) NOT NULL,
  `username` varchar(150) NOT NULL,
  `first_name` varchar(30) NOT NULL,
  `last_name` varchar(30) NOT NULL,
  `email` varchar(254) NOT NULL,
  `is_staff` tinyint(1) NOT NULL,
  `is_active` tinyint(1) NOT NULL,
  `date_joined` datetime(6) NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `username` (`username`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8;

-- ----------------------------
-- Records of auth_user
-- ----------------------------

-- ----------------------------
-- Table structure for auth_user_groups
-- ----------------------------
DROP TABLE IF EXISTS `auth_user_groups`;
CREATE TABLE `auth_user_groups` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `user_id` int(11) NOT NULL,
  `group_id` int(11) NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `auth_user_groups_user_id_group_id_94350c0c_uniq` (`user_id`,`group_id`),
  KEY `auth_user_groups_group_id_97559544_fk_auth_group_id` (`group_id`),
  CONSTRAINT `auth_user_groups_group_id_97559544_fk_auth_group_id` FOREIGN KEY (`group_id`) REFERENCES `auth_group` (`id`),
  CONSTRAINT `auth_user_groups_user_id_6a12ed8b_fk_auth_user_id` FOREIGN KEY (`user_id`) REFERENCES `auth_user` (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8;

-- ----------------------------
-- Records of auth_user_groups
-- ----------------------------

-- ----------------------------
-- Table structure for auth_user_user_permissions
-- ----------------------------
DROP TABLE IF EXISTS `auth_user_user_permissions`;
CREATE TABLE `auth_user_user_permissions` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `user_id` int(11) NOT NULL,
  `permission_id` int(11) NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `auth_user_user_permissions_user_id_permission_id_14a6b632_uniq` (`user_id`,`permission_id`),
  KEY `auth_user_user_permi_permission_id_1fbb5f2c_fk_auth_perm` (`permission_id`),
  CONSTRAINT `auth_user_user_permi_permission_id_1fbb5f2c_fk_auth_perm` FOREIGN KEY (`permission_id`) REFERENCES `auth_permission` (`id`),
  CONSTRAINT `auth_user_user_permissions_user_id_a95ead1b_fk_auth_user_id` FOREIGN KEY (`user_id`) REFERENCES `auth_user` (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8;

-- ----------------------------
-- Records of auth_user_user_permissions
-- ----------------------------

-- ----------------------------
-- Table structure for cirrnainfo_cirrna
-- ----------------------------
DROP TABLE IF EXISTS `cirrnainfo_cirrna`;
CREATE TABLE `cirrnainfo_cirrna` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `name_cirrna` varchar(500) DEFAULT NULL,
  `chrom` varchar(500) DEFAULT NULL,
  `start` varchar(500) DEFAULT NULL,
  `end` varchar(500) DEFAULT NULL,
  `strand` varchar(500) DEFAULT NULL,
  `circRNA_ID` varchar(500) DEFAULT NULL,
  `genomic_length` varchar(500) DEFAULT NULL,
  `spliced_seq_length` varchar(500) DEFAULT NULL,
  `samples` varchar(500) DEFAULT NULL,
  `repeats` varchar(500) DEFAULT NULL,
  `annotation` varchar(500) DEFAULT NULL,
  `best_transcript` varchar(500) DEFAULT NULL,
  `gene_symbol` varchar(500) DEFAULT NULL,
  `circRNA_study` varchar(500) DEFAULT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=33 DEFAULT CHARSET=utf8;

-- ----------------------------
-- Records of cirrnainfo_cirrna
-- ----------------------------
INSERT INTO `cirrnainfo_cirrna` VALUES ('1', 'Hsa_circ_0001649', 'chr6', '146209155', '146216113', '-', 'hsa_circ_0001649', '6958', '440', 'cd_19, Hs68_RNase, Hs68_control, K562, H1hesc, Gm12878, Ag04450, A549, Nhek, Sknshra', 'None', 'ANNOTATED, CDS, coding, INTERNAL, OVCODE, OVEXON', 'NM_173082', 'SHPRH', 'Jeck2013, Memczak2013, Salzman2013\n');
INSERT INTO `cirrnainfo_cirrna` VALUES ('2', 'hsa_circ_0005075', 'chr1', '21377358', '21415706', '-', 'hsa_circ_0005075', '38348', '205', 'Hs68_RNase, Hs68_control, K562, Huvec, Hsmm, Hepg2, Helas3, H1hesc, Bj, Ag04450, A549, Nhlf, Sknshra, Mcf7', 'None', 'ANNOTATED, coding, INTERNAL, OVEXON, UTR5', 'NM_001198801', 'EIF4G3', 'Jeck2013, Salzman2013\n');
INSERT INTO `cirrnainfo_cirrna` VALUES ('3', 'hsa_circ_0067934', 'chr3', '170013698', '170015181', '+', 'hsa_circ_0067934', '1483', '170', 'Nhek, K562, H1hesc, Bj, A549, Sknshra', 'NA', 'ANNOTATED, CDS, coding, INTERNAL, OVCODE, OVEXON', 'NM_002740', 'PRKCI', 'Salzman2013\n');
INSERT INTO `cirrnainfo_cirrna` VALUES ('4', 'hsa_circ_0000069', 'chr1', '47745912', '47748131', '-', 'hsa_circ_0000069', '2219', '1084', 'HEK293, Hs68_RNase, Hs68_control, Nhek, K562, Huvec, Hmec, Hepg2, Helas3, H1hesc, Gm12878, Bj, Ag04450, A549, Sknshra, Mcf7', 'None', 'ANNOTATED, CDS, coding, INTERNAL, OVCODE, OVEXON', 'NM_003035', 'STIL', 'Jeck2013, Memczak2013, Salzman2013\n');
INSERT INTO `cirrnainfo_cirrna` VALUES ('5', 'Hsa_circ_0000096', 'chr1', '100515464', '100535241', '+', 'hsa_circ_0000096', '19777', '807', 'cd_19, Hs68_RNase, Hs68_control', 'None', 'ANNOTATED, CDS, coding, INTERNAL, OVCODE, OVEXON', 'NM_033055', 'HIAT1', 'Jeck2013, Memczak2013\n');
INSERT INTO `cirrnainfo_cirrna` VALUES ('6', 'hsa_circ_0000190', 'chr1', '224553580', '224559125', '+', 'hsa_circ_0000190', '5545', '254', 'cd_34', 'None', 'ANNOTATED, CDS, coding, INTERNAL', 'NM_014184', 'CNIH4', 'Memczak2013\n');
INSERT INTO `cirrnainfo_cirrna` VALUES ('7', 'hsa_circ_0005986', 'chr1', '14057494', '14068652', '+', 'hsa_circ_0005986', '11158', '375', 'Hs68_RNase, Hs68_control, K562, Gm12878, Bj, Ag04450, A549, Sknshra', 'None', 'ANNOTATED, CDS, coding, INTERNAL, OVCODE, OVEXON', 'NM_012231', 'PRDM2', 'Jeck2013, Salzman2013\n');
INSERT INTO `cirrnainfo_cirrna` VALUES ('8', 'hsa_circ_0001895', 'chr9', '134305476', '134308181', '+', 'hsa_circ_0001895', '2705', '348', 'cd_19, Hs68_RNase, Hs68_control, K562, Helas3, H1hesc, Gm12878, Bj, Ag04450, A549, Sknshra, Mcf7', 'None', 'ANNOTATED, CDS, coding, OVCODE, OVEXON, UTR5', 'NM_013318', 'PRRC2B', 'Jeck2013, Memczak2013, Salzman2013\n');
INSERT INTO `cirrnainfo_cirrna` VALUES ('9', 'hsa_circ_0003570', 'chr10', '126370175', '126384781', '-', 'hsa_circ_0003570', '14606', '828', 'Hs68_RNase, Hs68_control, Nhek, K562, Hsmm, Hepg2, Helas3, H1hesc, Gm12878, Bj, Ag04450, A549, Sknshra', 'None', 'ANNOTATED, CDS, coding, INTERNAL, OVCODE, OVEXON', 'NM_014661', 'FAM53B', 'Jeck2013, Salzman2013\n');
INSERT INTO `cirrnainfo_cirrna` VALUES ('10', 'Hsa_circ_0014717', 'chr1', '156290629', '156304709', '-', 'hsa_circ_0014717', '14080', '516', 'K562, H1hesc, Gm12878, A549', 'NA', 'ANNOTATED, INTERNAL, ncRNA, OVCODE, OVEXON', 'NR_036564', 'CCT3', 'Salzman2013\n');
INSERT INTO `cirrnainfo_cirrna` VALUES ('11', 'hsa_circ_0010729', 'chr1', '22186058', '22207328', '-', 'hsa_circ_0010729', '21270', '3475', 'Huvec', 'NA', 'ANNOTATED, CDS, coding, INTERNAL, OVCODE, OVERLAPTX, OVEXON', 'NM_005529', 'HSPG2', 'Salzman2013\n');
INSERT INTO `cirrnainfo_cirrna` VALUES ('12', 'hsa_circ_0003159', 'chr7', '81689743', '81746489', '-', 'hsa_circ_0003159', '56746', '483', 'Hs68_RNase, Hs68_control, Hepg2, Gm12878, A549, Sknshra', 'None', 'ANNOTATED, CDS, coding, INTERNAL, OVCODE, OVEXON', 'NM_000722', 'CACNA2D1', 'Jeck2013, Salzman2013\n');
INSERT INTO `cirrnainfo_cirrna` VALUES ('13', 'hsa_circ_0058246', 'chr2', '219305444', '219305585', '+', 'hsa_circ_0058246', '141', '141', 'Helas3', 'NA', 'ANNOTATED, CDS, coding, INTERNAL, OVCODE, OVEXON', 'NM_007127', 'VIL1', 'Salzman2013\n');
INSERT INTO `cirrnainfo_cirrna` VALUES ('14', 'hsa_circ_0005402', 'chr15', '60648117', '60674640', '-', 'hsa_circ_0005402', '26523', '480', 'Hs68_RNase, Hs68_control, Nhek, Huvec, Bj, Ag04450, A549, Nhlf, Sknshra', 'None', 'ANNOTATED, CDS, coding, INTERNAL, OVCODE, OVEXON', 'NM_001136015', 'ANXA2', 'Jeck2013, Salzman2013\n');
INSERT INTO `cirrnainfo_cirrna` VALUES ('15', 'hsa_circ_0035560', 'chr15', '60653139', '60674640', '-', 'hsa_circ_0035560', '21501', '309', 'Nhek, K562, Huvec, Hsmm, Hmec, Hepg2, Helas3, Bj, Ag04450, A549, Nhlf, Sknshra', 'NA', 'ANNOTATED, CDS, coding, INTERNAL, OVCODE, OVERLAPTX, OVEXON', 'NM_001136015', 'ANXA2', 'Salzman2013\n');
INSERT INTO `cirrnainfo_cirrna` VALUES ('16', 'hsa_circ_0006633', 'chr1', '59805629', '59844509', '+', 'hsa_circ_0006633', '38880', '353', 'Hs68_RNase, Hs68_control, Nhek, K562, H1hesc, Gm12878, Ag04450', 'None', 'ANNOTATED, CDS, coding, INTERNAL, OVCODE, OVEXON', 'NM_001113411', 'FGGY', 'Jeck2013, Salzman2013\n');
INSERT INTO `cirrnainfo_cirrna` VALUES ('17', 'hsa_circ_0013958', 'chr1', '147131074', '147131890', '-', 'hsa_circ_0013958', '816', '340', 'Helas3, H1hesc, Mcf7', 'NA', 'ANNOTATED, CDS, coding, INTERNAL, OVCODE, OVEXON', 'NM_016361', 'ACP6', 'Salzman2013\n');
INSERT INTO `cirrnainfo_cirrna` VALUES ('18', 'Hsa_circ_0020397', 'chr10', '128768965', '128926028', '+', 'hsa_circ_0020397', '157063', '2738', 'Nhek, Hmec, Hepg2, Helas3, H1hesc, Ag04450, Sknshra, Mcf7', 'NA', 'ANNOTATED, CDS, coding, INTERNAL, OVCODE, OVERLAPTX, OVEXON', 'NM_001380', 'DOCK1', 'Salzman2013\n');
INSERT INTO `cirrnainfo_cirrna` VALUES ('19', 'hsa_circ_0071410', 'chr4', '169812072', '169837178', '+', 'hsa_circ_0071410', '25106', '886', 'Hsmm, A549', 'NA', 'ANNOTATED, CDS, coding, INTERNAL, OVCODE, OVERLAPTX, OVEXON', 'NM_001166108', 'PALLD', 'Salzman2013\n');
INSERT INTO `cirrnainfo_cirrna` VALUES ('20', 'hsa_circ_0045714', 'chr17', '73808192', '73809959', '+', 'hsa_circ_0045714', '1767', '562', 'Nhek, Hsmm, Helas3, H1hesc, Bj, Ag04450, Sknshra, Mcf7', 'NA', 'ANNOTATED, CDS, coding, INTERNAL, OVCODE, OVERLAPTX, OVEXON', 'NM_001080419', 'UNK', 'Salzman2013\n');
INSERT INTO `cirrnainfo_cirrna` VALUES ('21', 'hsa_circ_0006528', 'chr5', '145197456', '145205763', '-', 'hsa_circ_0006528', '8307', '496', 'Hs68_RNase, Hs68_control, K562, Hepg2, Helas3, H1hesc, Bj, Ag04450, A549, Sknshra, Mcf7', 'None', 'ANNOTATED, CDS, coding, INTERNAL, OVCODE, OVEXON, UTR5', 'NM_138492', 'PRELID2', 'Jeck2013, Salzman2013\n');
INSERT INTO `cirrnainfo_cirrna` VALUES ('22', 'hsa_circ_0005836', 'chr14', '39623414', '39627606', '-', 'hsa_circ_0005836', '4192', '202', 'Hs68_RNase, Hs68_control, K562, Helas3, Gm12878, Bj, Ag04450', 'None', 'ANNOTATED, CDS, coding, INTERNAL, OVCODE, OVEXON', 'NM_001079537', 'TRAPPC6B', 'Jeck2013, Salzman2013\n');
INSERT INTO `cirrnainfo_cirrna` VALUES ('23', 'hsa_circ_0001982', 'chr15', '59323002', '59323901', '+', 'hsa_circ_0001982', '899', '899', 'Hs68_RNase, Hs68_control, Nhek, K562, Hepg2, Ag04450, A549, Sknshra', 'None', 'ANNOTATED, CDS, coding, INTERNAL, OVCODE, OVEXON, UTR5', 'NM_017610', 'RNF111', 'Jeck2013, Salzman2013\n');
INSERT INTO `cirrnainfo_cirrna` VALUES ('24', 'hsa_circ_0000181', 'chr1', '212977661', '212981190', '+', 'hsa_circ_0000181', '3529', '279', 'cd_34, K562', 'None', 'ANNOTATED, CDS, coding, INTERNAL', 'NM_001146171', 'TATDN3', 'Memczak2013, Salzman2013\n');
INSERT INTO `cirrnainfo_cirrna` VALUES ('25', 'hsa_circ_0003575', 'chr9', '33271149', '33278223', '+', 'hsa_circ_0003575', '7074', '294', 'Hs68_RNase, Hs68_control', 'None', 'ANNOTATED, CDS, coding, INTERNAL, OVCODE, OVEXON', 'NM_016410', 'CHMP5', 'Jeck2013\n');
INSERT INTO `cirrnainfo_cirrna` VALUES ('26', 'hsa_circ_0043256', 'chr17', '35604934', '35609962', '-', 'hsa_circ_0043256', '5028', '483', 'K562, Hepg2, Helas3, Gm12878, A549, Sknshra', 'NA', 'ANNOTATED, CDS, coding, INTERNAL, OVCODE, OVEXON', 'NM_198839', 'ACACA', 'Salzman2013\n');
INSERT INTO `cirrnainfo_cirrna` VALUES ('27', 'hsa_circ_0000745', 'chr17', '20107645', '20109225', '+', 'hsa_circ_0000745', '1580', '1580', 'HEK293, Nhek, K562, Huvec, Hepg2, Helas3, H1hesc, Gm12878, Bj, Ag04450, A549, Sknshra, Mcf7', 'None', 'ANNOTATED, CDS, coding, INTERNAL', 'NM_001033554', 'SPECC1', 'Memczak2013, Salzman2013\n');
INSERT INTO `cirrnainfo_cirrna` VALUES ('28', 'hsa_circ_0047905', 'chr18', '61156579', '61172318', '+', 'hsa_circ_0047905', '15739', '2185', 'Nhek', 'NA', 'ANNOTATED, CDS, coding, OVCODE, OVEXON, UTR3', 'NM_002639', 'SERPINB5', 'Salzman2013\n');
INSERT INTO `cirrnainfo_cirrna` VALUES ('29', 'hsa_circ_0001785', 'chr8', '28013458', '28019595', '+', 'hsa_circ_0001785', '6137', '467', 'cd_19, Hs68_RNase, Hs68_control, Nhek, K562, Huvec, Hepg2, Helas3, Gm12878, Ag04450, Sknshra', 'None', 'ANNOTATED, CDS, coding, INTERNAL, OVCODE, OVEXON', 'NM_018091', 'ELP3', 'Jeck2013, Memczak2013, Salzman2013\n');
INSERT INTO `cirrnainfo_cirrna` VALUES ('30', 'Hsa_circ_0000520', 'chr14', '20811436', '20811559', '-', 'hsa_circ_0000520', '123', '123', 'cd_19, neutr, Hs68_RNase, Hs68_control', 'None', 'ALT_ACCEPTOR, ALT_DONOR, INTERNAL, ncRNA, OVEXON', 'NR_002312', 'RPPH1', 'Jeck2013, Memczak2013\n');
INSERT INTO `cirrnainfo_cirrna` VALUES ('31', 'Hsa_circ_0001017', 'chr2', '61749745', '61761038', '-', 'hsa_circ_0001017', '11293', '307', 'HEK293, cd_19, neutr, Hs68_RNase, Hs68_control, Nhek, K562, Huvec, Hepg2, Helas3, H1hesc, Gm12878, Bj, Ag04450, A549, Sknshra', 'None', 'ANNOTATED, CDS, coding, INTERNAL, OVCODE, OVEXON, UTR5', 'NM_003400', 'XPO1', 'Jeck2013, Memczak2013, Salzman2013\n');
INSERT INTO `cirrnainfo_cirrna` VALUES ('32', 'hsa_circ_0061276', 'chr21', '16415815', '16415895', '-', 'hsa_circ_0061276', '80', '80', 'K562, Huvec, Hepg2, Helas3, H1hesc, Gm12878, Bj, Ag04450, A549', 'NA', 'ANNOTATED, coding, INTERNAL, OVEXON, UTR5', 'NM_003489', 'NRIP1', 'Salzman2013\n');

-- ----------------------------
-- Table structure for cirrnainfo_disease
-- ----------------------------
DROP TABLE IF EXISTS `cirrnainfo_disease`;
CREATE TABLE `cirrnainfo_disease` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `name_incirrna` varchar(500) DEFAULT NULL,
  `name` varchar(500) DEFAULT NULL,
  `DOID` longtext,
  `Definition` longtext,
  `Xrefs` longtext,
  `Alternateids` longtext,
  `Subsets` longtext,
  `Synonyms` longtext,
  `Relationships` longtext,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=28 DEFAULT CHARSET=utf8;

-- ----------------------------
-- Records of cirrnainfo_disease
-- ----------------------------
INSERT INTO `cirrnainfo_disease` VALUES ('1', 'gastric carcinoma', 'stomach carcinoma', ' DOID:5517', ' \"A stomach cancer that is located_in the stomach.\" [url:http://en.wikipedia.org/wiki/Stomach_cancer]', ' EFO:0000178<> NCI:C4911<> SNOMEDCT_US_2016_03_01:154446008<> SNOMEDCT_US_2016_03_01:255080008<> SNOMEDCT_US_2016_03_01:372143007<> UMLS_CUI:C0699791', '', '', ' \"cancer of the stomach\" EXACT [NCI2004_11_17:C4911]<> \"carcinoma of stomach\" EXACT [SNOMEDCT_2005_07_31:154446008]<> \"carcinoma of stomach (disorder)\" EXACT [SNOMEDCT_2005_07_31:255080008]<> \"carcinoma of stomach (disorder)\" EXACT [SNOMEDCT_2005_07_31:372143007]<>\"gastric carcinoma\" EXACT []', ' DOID:10534 ! stomach cancer');
INSERT INTO `cirrnainfo_disease` VALUES ('2', 'hepatocellular carcinoma', 'hepatocellular carcinoma', ' DOID:684', ' \"A liver carcinoma that has_material_basis_in undifferentiated hepatocytes.\" [url:http://cancergenome.nih.gov/cancersselected/LiverHepatocellularCarcinoma, url:http://en.wikipedia.org/wiki/Hepatocellular_carcinoma, url:http://www.omim.org/entry/114550]', ' EFO:0000182<> NCI:C7711<> NCI:C7956<> OMIM:114550<> UMLS_CUI:C0279607', ' DOID:5005', 'DO_cancer_slim', ' \"adult Hepatoma\" EXACT [NCI2004_11_17:C7956]<>\"adult primary hepatocellular carcinoma\" EXACT []<> \"HCC\" EXACT [NCI2004_11_17:C3099]<> \"Hepatoma\" EXACT [SNOMEDCT_2005_07_31:126851005]', ' DOID:686 ! liver carcinoma');
INSERT INTO `cirrnainfo_disease` VALUES ('3', 'pre-eclampsia', 'pre-eclampsia', ' DOID:10591', ' \"A hypertension occurring during pregnancy characterized by large amounts of protein in the urine (proteinuria) and edema, usually by the last trimester of pregnancy.\" [url:http://en.wikipedia.org/wiki/Pre-eclampsia, url:http://ghr.nlm.nih.gov/condition/preeclampsia, url:http://www.ncbi.nlm.nih.gov/pubmed/?term=24400024]', ' ICD10CM:O14<> ICD10CM:O14.9<> ICD10CM:O14.90<> MESH:D011225<> NCI:C34943<> NCI:C85021<> OMIM:189800<> OMIM:609402<> OMIM:609403<> OMIM:609404<> OMIM:614592<> ORDO:275555<> SNOMEDCT_US_2016_03_01:15394000<> SNOMEDCT_US_2016_03_01:156106005<> SNOMEDCT_US_2016_03_01:156109003<> SNOMEDCT_US_2016_03_01:198972006<> SNOMEDCT_US_2016_03_01:198979002<> SNOMEDCT_US_2016_03_01:199011002<> SNOMEDCT_US_2016_03_01:237280005<> SNOMEDCT_US_2016_03_01:267306006<> SNOMEDCT_US_2016_03_01:288201007<> SNOMEDCT_US_2016_03_01:398254007<> SNOMEDCT_US_2016_03_01:46764007<> SNOMEDCT_US_2016_03_01:6758009<> UMLS_CUI:C0032914', ' DOID:12684', 'DO_rare_slim', ' \"gestational hypertension\" EXACT [MTHICD9_2006:642.3, SNOMEDCT_2005_07_31:30354006]<>\"hypertension induced by pregnancy\" EXACT []<> \"pre-eclamptic toxaemia\" EXACT [SNOMEDCT_2005_07_31:46764007]<> \"preeclampsia\" EXACT [CSP2005:4001-0110]<>\"preeclampsia/eclampsia\" EXACT []<> \"pregnancy associated hypertension\" EXACT [NCI2004_11_17:C4371]<> \"pregnancy toxemia\" EXACT [CSP2005:2404-7447]<> \"proteinuric hypertension of pregnancy\" EXACT [SNOMEDCT_2005_07_31:199011002]<> \"toxaemia of pregnancy\" EXACT [SNOMEDCT_2005_07_31:156106005]', ' DOID:10763 ! hypertension');
INSERT INTO `cirrnainfo_disease` VALUES ('4', 'colorectal cancer', 'colorectal cancer', ' DOID:9256', ' \"A large intestine cancer that is located_in the colon and/or located_in the rectum.\" [url:http://www.cancer.gov/dictionary?CdrID=444983]', ' KEGG:05210<> MESH:D015179<> OMIM:114500<> OMIM:611469<> OMIM:612230<> OMIM:612231<> OMIM:612232<> OMIM:612589<> OMIM:612590<> OMIM:612592<> UMLS_CUI:C1527249', '', 'DO_FlyBase_slim', '', ' DOID:5672 ! large intestine cancer');
INSERT INTO `cirrnainfo_disease` VALUES ('5', 'osteoarthritis', 'osteoarthritis', ' DOID:8398', ' \"An arthritis that has_material_basis_in worn out cartilage located_in joint.\" [url:http://en.wikipedia.org/wiki/Osteoarthritis, url:http://www.mayoclinic.com/health/osteoarthritis/DS00019, url:http://www.medicinenet.com/osteoarthritis/article.htm, url:http://www.nlm.nih.gov/medlineplus/ency/article/000423.htm]', ' EFO:0002506<> ICD9CM:715.3<> OMIM:610839<> OMIM:612401<> UMLS_CUI:C0157946', '', '', '\"degenerative arthritis\" EXACT []<>\"degenerative joint disease\" EXACT []<>\"hypertrophic arthritis\" EXACT []<>\"osteoarthrosis\" EXACT []<>\"Osteoarthrosis and allied disorder\" EXACT []', ' DOID:848 ! arthritis');
INSERT INTO `cirrnainfo_disease` VALUES ('6', 'major depressive disorder', 'major depressive disorder', ' DOID:1470', ' \"An endogenous depression that is characterized by an all-encompassing low mood accompanied by low self-esteem, and by loss of interest or pleasure in normally enjoyable activities.\" [url:http://en.wikipedia.org/wiki/Major_depressive_disorder]', ' ICD10CM:F32<> ICD10CM:F32.9<> ICD10CM:F33<> ICD10CM:F33.9<> ICD9CM:296.2<> ICD9CM:296.20<> ICD9CM:296.3<> ICD9CM:296.30<> MESH:D003865<> NCI:C34796<> OMIM:608520<> OMIM:608691<> SNOMEDCT_US_2016_03_01:191599006<> SNOMEDCT_US_2016_03_01:191600009<> SNOMEDCT_US_2016_03_01:191607007<> SNOMEDCT_US_2016_03_01:191608002<> SNOMEDCT_US_2016_03_01:191609005<> SNOMEDCT_US_2016_03_01:191617002<> SNOMEDCT_US_2016_03_01:192366006<> SNOMEDCT_US_2016_03_01:268620009<> SNOMEDCT_US_2016_03_01:268621008<> SNOMEDCT_US_2016_03_01:36923009<> SNOMEDCT_US_2016_03_01:66344007<> UMLS_CUI:C0024517<> UMLS_CUI:C0154409', ' DOID:10614<> DOID:2240<> DOID:2241<> DOID:7580', '', '\"recurrent major depression\" EXACT []<>\"single major depressive episode\" EXACT []', ' DOID:1596 ! mental depression');
INSERT INTO `cirrnainfo_disease` VALUES ('7', 'bladder carcinoma', 'bladder carcinoma', ' DOID:4007', ' \"A urinary bladder cancer that has_material_basis_in abnormally proliferating cells derives_from epithelial cells.\" [url:http://nci.nih.gov/cancertopics/types/bladder]', ' EFO:0000292<> NCI:C4912<> SNOMEDCT_US_2016_03_01:154540000<> SNOMEDCT_US_2016_03_01:255108000<> SNOMEDCT_US_2016_03_01:269607003<> UMLS_CUI:C0699885', '', 'DO_cancer_slim', ' \"carcinoma bladder\" EXACT [SNOMEDCT_2005_07_31:269607003]<> \"carcinoma bladder\" EXACT [SNOMEDCT_2005_07_31:154540000]<> \"carcinoma OF BLADDER\" EXACT [MTH:108]<>\"carcinoma of bladder\" RELATED []<> \"carcinoma of bladder (disorder)\" EXACT [SNOMEDCT_2005_07_31:255108000]<> \"carcinoma of urinary bladder\" EXACT [NCI2004_11_17:C4912]', ' DOID:11054 ! urinary bladder cancer');
INSERT INTO `cirrnainfo_disease` VALUES ('8', 'preeclampsia', 'pre-eclampsia', ' DOID:10591', ' \"A hypertension occurring during pregnancy characterized by large amounts of protein in the urine (proteinuria) and edema, usually by the last trimester of pregnancy.\" [url:http://en.wikipedia.org/wiki/Pre-eclampsia, url:http://ghr.nlm.nih.gov/condition/preeclampsia, url:http://www.ncbi.nlm.nih.gov/pubmed/?term=24400024]', ' ICD10CM:O14<> ICD10CM:O14.9<> ICD10CM:O14.90<> MESH:D011225<> NCI:C34943<> NCI:C85021<> OMIM:189800<> OMIM:609402<> OMIM:609403<> OMIM:609404<> OMIM:614592<> ORDO:275555<> SNOMEDCT_US_2016_03_01:15394000<> SNOMEDCT_US_2016_03_01:156106005<> SNOMEDCT_US_2016_03_01:156109003<> SNOMEDCT_US_2016_03_01:198972006<> SNOMEDCT_US_2016_03_01:198979002<> SNOMEDCT_US_2016_03_01:199011002<> SNOMEDCT_US_2016_03_01:237280005<> SNOMEDCT_US_2016_03_01:267306006<> SNOMEDCT_US_2016_03_01:288201007<> SNOMEDCT_US_2016_03_01:398254007<> SNOMEDCT_US_2016_03_01:46764007<> SNOMEDCT_US_2016_03_01:6758009<> UMLS_CUI:C0032914', ' DOID:12684', 'DO_rare_slim', ' \"gestational hypertension\" EXACT [MTHICD9_2006:642.3, SNOMEDCT_2005_07_31:30354006]<>\"hypertension induced by pregnancy\" EXACT []<> \"pre-eclamptic toxaemia\" EXACT [SNOMEDCT_2005_07_31:46764007]<> \"preeclampsia\" EXACT [CSP2005:4001-0110]<>\"preeclampsia/eclampsia\" EXACT []<> \"pregnancy associated hypertension\" EXACT [NCI2004_11_17:C4371]<> \"pregnancy toxemia\" EXACT [CSP2005:2404-7447]<> \"proteinuric hypertension of pregnancy\" EXACT [SNOMEDCT_2005_07_31:199011002]<> \"toxaemia of pregnancy\" EXACT [SNOMEDCT_2005_07_31:156106005]', ' DOID:10763 ! hypertension');
INSERT INTO `cirrnainfo_disease` VALUES ('9', 'lung cancer', 'lung cancer', ' DOID:1324', ' \"A respiratory system cancer that is located_in the lung.\" [url:http://en.wikipedia.org/wiki/Lung_cancer]', ' ICD10CM:C34.1<> ICD10CM:C34.2<> ICD10CM:C34.3<> ICD9CM:162.3<> ICD9CM:162.4<> ICD9CM:162.5<> ICD9CM:162.8<> OMIM:211980<> OMIM:608935<> OMIM:612571<> OMIM:612593<> OMIM:614210<> SNOMEDCT_US_2016_03_01:187860004<> SNOMEDCT_US_2016_03_01:187863002<> SNOMEDCT_US_2016_03_01:187864008<> SNOMEDCT_US_2016_03_01:187867001<> SNOMEDCT_US_2016_03_01:187868006<> SNOMEDCT_US_2016_03_01:187871003<> SNOMEDCT_US_2016_03_01:187874006<> SNOMEDCT_US_2016_03_01:269464000<> UMLS_CUI:C0024624<> UMLS_CUI:C0153491<> UMLS_CUI:C0153492<> UMLS_CUI:C0153493', ' DOID:13075<> DOID:1322<> DOID:9881', 'TopNodes_DOcancerslim', '\"lung neoplasm\" RELATED []', ' DOID:0050615 ! respiratory system cancer');
INSERT INTO `cirrnainfo_disease` VALUES ('10', 'Hirschsprung disease', 'Hirschsprung^s disease', ' DOID:10487', ' \"A megacolon that is characterized by a blockage of the large intestine due to improper muscle movement in the bowel.\" [url:http://digestive.niddk.nih.gov/ddiseases/pubs/hirschsprungs_ez/, url:http://en.wikipedia.org/wiki/Hirschsprung%27s_disease, url:http://www.nlm.nih.gov/medlineplus/ency/article/001140.htm, url:http://www.uptodate.com/contents/congenital-aganglionic-megacolon-hirschsprung-disease]', ' ICD10CM:Q43.1<> MESH:D006627<> NCI:C34700<> OMIM:600156<> OMIM:606874<> OMIM:606875<> OMIM:608462<> OMIM:611644<> ORDO:388<> SNOMEDCT_US_2016_03_01:156957008<> SNOMEDCT_US_2016_03_01:204738000<> SNOMEDCT_US_2016_03_01:204739008<> SNOMEDCT_US_2016_03_01:204742002<> SNOMEDCT_US_2016_03_01:204744001<> SNOMEDCT_US_2016_03_01:360436002<> SNOMEDCT_US_2016_03_01:367495003<> SNOMEDCT_US_2016_03_01:6687001<> UMLS_CUI:C0019569', '', 'DO_rare_slim', ' \"aganglionic megacolon\" EXACT [NCI2004_11_17:C34700]<>\"congenital megacolon\" EXACT []<>\"Hirschsprung disease\" EXACT []<> \"macrocolon\" EXACT [SNOMEDCT_2005_07_31:204744001]<> \"pelvirectal achalasia\" EXACT [CSP2005:0724-7702]<>\"total intestinal aganglionosis\" EXACT []', ' DOID:11372 ! megacolon');
INSERT INTO `cirrnainfo_disease` VALUES ('11', 'coronary artery disease', 'coronary artery disease', ' DOID:3393', ' \"An artery disease that is characterized by plaque building up along the inner walls of the arteries of the heart resulting in a narrowing of the arteries and a reduced blood supply to the cardiac muscles.\" [url:http://en.wikipedia.org/wiki/Coronary_heart_disease]', ' EFO:0001645<> ICD10CM:I20-I25<> ICD10CM:I25<> ICD10CM:I25.1<> ICD10CM:I25.10<> ICD10CM:I25.9<> ICD10CM:K76.1<> ICD9CM:410-414.99<> ICD9CM:414.0<> ICD9CM:414.9<> MESH:D003324<> MESH:D003327<> MESH:D017202<> NCI:C35505<> NCI:C50625<> OMIM:300464<> OMIM:607339<> OMIM:608316<> OMIM:608318<> OMIM:608320<> OMIM:610947<> OMIM:611139<> OMIM:612030<> OMIM:614293<> SNOMEDCT_US_2016_03_01:155303000<> SNOMEDCT_US_2016_03_01:155315001<> SNOMEDCT_US_2016_03_01:155316000<> SNOMEDCT_US_2016_03_01:155318004<> SNOMEDCT_US_2016_03_01:155322009<> SNOMEDCT_US_2016_03_01:194795001<> SNOMEDCT_US_2016_03_01:194841001<> SNOMEDCT_US_2016_03_01:194852007<> SNOMEDCT_US_2016_03_01:194878003<> SNOMEDCT_US_2016_03_01:195540001<> SNOMEDCT_US_2016_03_01:233822007<> SNOMEDCT_US_2016_03_01:2610009<> SNOMEDCT_US_2016_03_01:266231003<> SNOMEDCT_US_2016_03_01:266290000<> SNOMEDCT_US_2016_03_01:266291001<> SNOMEDCT_US_2016_03_01:271430002<> SNOMEDCT_US_2016_03_01:32598000<> SNOMEDCT_US_2016_03_01:413838009<> SNOMEDCT_US_2016_03_01:413844008<> SNOMEDCT_US_2016_03_01:414545008<> SNOMEDCT_US_2016_03_01:414795007<> SNOMEDCT_US_2016_03_01:41702007<> SNOMEDCT_US_2016_03_01:443502000<> SNOMEDCT_US_2016_03_01:53741008<> SNOMEDCT_US_2016_03_01:84537008<> UMLS_CUI:C0010054<> UMLS_CUI:C0010068<> UMLS_CUI:C0151744<> UMLS_CUI:C0264694', ' DOID:10506<> DOID:3363<> DOID:3394<> DOID:9420', '', ' \"CHD (coronary heart disease)\" EXACT [CSP2005:1393-3397]<>\"coronary arteriosclerosis\" EXACT []<> \"Coronary disease\" EXACT [NCI2004_11_17:C26732]<>\"coronary heart disease\" EXACT []', ' DOID:0050828 ! artery disease');
INSERT INTO `cirrnainfo_disease` VALUES ('12', 'gastric cancer', 'stomach cancer', ' DOID:10534', ' \"A gastrointestinal system cancer that is located_in the stomach.\" [url:http://cancergenome.nih.gov/cancersselected/stomach-esophagealcancer, url:http://en.wikipedia.org/wiki/Stomach]', ' ICD10CM:C16<> ICD10CM:C16.2<> ICD10CM:C16.5<> ICD10CM:C16.6<> ICD10CM:C16.9<> ICD9CM:151<> ICD9CM:151.4<> ICD9CM:151.5<> ICD9CM:151.6<> ICD9CM:151.9<> MESH:D013274<> NCI:C3387<> NCI:C9331<> OMIM:137215<> OMIM:613659<> SNOMEDCT_US_2016_03_01:126824007<> SNOMEDCT_US_2016_03_01:154450001<> SNOMEDCT_US_2016_03_01:154451002<> SNOMEDCT_US_2016_03_01:154452009<> SNOMEDCT_US_2016_03_01:154453004<> SNOMEDCT_US_2016_03_01:187731004<> SNOMEDCT_US_2016_03_01:187742008<> SNOMEDCT_US_2016_03_01:187743003<> SNOMEDCT_US_2016_03_01:187744009<> SNOMEDCT_US_2016_03_01:187750004<> SNOMEDCT_US_2016_03_01:255080008<> SNOMEDCT_US_2016_03_01:269458007<> SNOMEDCT_US_2016_03_01:269459004<> SNOMEDCT_US_2016_03_01:269460009<> SNOMEDCT_US_2016_03_01:269529007<> SNOMEDCT_US_2016_03_01:269530002<> SNOMEDCT_US_2016_03_01:269531003<> SNOMEDCT_US_2016_03_01:269532005<> SNOMEDCT_US_2016_03_01:363349007<> SNOMEDCT_US_2016_03_01:93717002<> SNOMEDCT_US_2016_03_01:93818001<> SNOMEDCT_US_2016_03_01:93867004<> SNOMEDCT_US_2016_03_01:94074003<> UMLS_CUI:C0024623<> UMLS_CUI:C0038356<> UMLS_CUI:C0153421<> UMLS_CUI:C0153422<> UMLS_CUI:C0153423', ' DOID:10539<> DOID:10542<> DOID:10543<> DOID:4713', 'DO_cancer_slim<>DO_FlyBase_slim<>TopNodes_DOcancerslim', ' \"Ca body - stomach\" EXACT [SNOMEDCT_2005_07_31:154450001]<>\"ca greater curvature of stomach\" EXACT []<> \"Ca lesser curvature - stomach (disorder)\" EXACT [SNOMEDCT_2005_07_31:269530002]<>\"gastric cancer\" EXACT []<> \"gastric neoplasm\" EXACT [CSP2005:2010-1611, NCI2004_11_17:C3387, SNOMEDCT_2005_07_31:187731004]<>\"malignant neoplasm of body of stomach\" EXACT []<> \"malignant neoplasm of lesser curve of stomach unspecified (disorder)\" EXACT [SNOMEDCT_2005_07_31:187743003]<> \"malignant tumor of body of stomach (disorder)\" EXACT [SNOMEDCT_2005_07_31:187742008]<> \"malignant tumor of greater curve of stomach (disorder)\" EXACT [SNOMEDCT_2005_07_31:269460009]<> \"malignant tumor of lesser curve of stomach (disorder)\" EXACT [SNOMEDCT_2005_07_31:269459004]', ' DOID:3119 ! gastrointestinal system cancer');
INSERT INTO `cirrnainfo_disease` VALUES ('13', 'Duchenne muscular dystrophy', 'Duchenne muscular dystrophy', ' DOID:11723', ' \"A muscular dystrophy that has_material_basis_in X-linked disease that has material basis in mutations in the DMD gene found on the X chromosome. It is characterized by rapidly progressing muscle weakness and muscle atrophy initially involving the lower extremities and eventually affecting the whole body. It affects males whereas females can be carriers. The symptoms start before the age of six and may appear at infancy.\" [url:http://en.wikipedia.org/wiki/Duchenne_muscular_dystrophy, url:http://omim.org/entry/300377, url:http://www.genome.gov/19518854]', ' MESH:D020388<> NCI:C75482<> OMIM:310200<> SNOMEDCT_US_2016_03_01:129619006<> SNOMEDCT_US_2016_03_01:155095006<> SNOMEDCT_US_2016_03_01:267712004<> SNOMEDCT_US_2016_03_01:387732009<> SNOMEDCT_US_2016_03_01:76670001<> UMLS_CUI:C0013264', '', 'DO_FlyBase_slim', ' \"Muscular dystrophy, Duchenne\" EXACT [MTHICD9_2006:359.1]', ' DOID:9884 ! muscular dystrophy');
INSERT INTO `cirrnainfo_disease` VALUES ('14', 'osteosarcoma', 'osteosarcoma', ' DOID:3347', ' \"A bone cancer that is located_in bone that has_material_basis_in cells of mesenchymal origin.\" [url:http://en.wikipedia.org/wiki/Osteosarcoma, url:http://www.cancer.gov/dictionary?cdrid=45395]', ' MESH:D012516<> MESH:D018213<> NCI:C9145<> OMIM:259500<> SNOMEDCT_US_2016_03_01:189878003<> SNOMEDCT_US_2016_03_01:21708004<> SNOMEDCT_US_2016_03_01:307576001<> SNOMEDCT_US_2016_03_01:408387006<> UMLS_CUI:C0029463<> UMLS_CUI:C0206639', ' DOID:183', 'DO_cancer_slim', '\"bone tissue neoplasm\" EXACT []<> \"Osteogenic sarcoma\" EXACT [NCI2004_11_17:C9145]<> \"osteoid sarcoma\" EXACT [CSP2005:2019-1578]<> \"Skeletal sarcoma\" EXACT [NCI2004_11_17:C9312]', ' DOID:184 ! bone cancer');
INSERT INTO `cirrnainfo_disease` VALUES ('15', 'myocardial infarction', 'myocardial infarction', ' DOID:5844', ' \"A coronary artery disease characterized by myocardial cell death (myocardial necrosis) due to prolonged ischaemia.\" [url:http://www.ncbi.nlm.nih.gov/pubmed/?term=17951287, url:https://en.wikipedia.org/wiki/Myocardial_infarction]', ' EFO:0000612<> ICD10CM:I21<> ICD10CM:I22<> MESH:D009203<> NCI:C27996<> OMIM:608557<> SNOMEDCT_US_2016_03_01:155304006<> SNOMEDCT_US_2016_03_01:194796000<> SNOMEDCT_US_2016_03_01:22298006<> SNOMEDCT_US_2016_03_01:233824008<> SNOMEDCT_US_2016_03_01:266288001<> SNOMEDCT_US_2016_03_01:66514008<> UMLS_CUI:C0027051', '', '', ' \"heart attack\" EXACT [CSP2005:1393-3417]<> \"Myocardial infarct\" EXACT [SNOMEDCT_2005_07_31:266288001]', ' DOID:3393 ! coronary artery disease');
INSERT INTO `cirrnainfo_disease` VALUES ('16', 'cardiovascular system disease', 'cardiovascular system disease', ' DOID:1287', ' \"A disease of anatomical entity which occurs in the blood, heart, blood vessels or the lymphatic system that passes nutrients (such as amino acids and electrolytes), gases, hormones, blood cells or lymph to and from cells in the body to help fight diseases and help stabilize body temperature and pH to maintain homeostasis.\" [url:http://en.wikipedia.org/wiki/Circulatory_system]', ' ICD9CM:429.2<> MESH:D002318<> NCI:C2931<> SNOMEDCT_US_2016_03_01:155263000<> SNOMEDCT_US_2016_03_01:194707003<> SNOMEDCT_US_2016_03_01:195139006<> SNOMEDCT_US_2016_03_01:195594006<> SNOMEDCT_US_2016_03_01:266275004<> SNOMEDCT_US_2016_03_01:266336005<> SNOMEDCT_US_2016_03_01:49601007<> UMLS_CUI:C0007222', ' DOID:73', 'DO_AGR_slim<>DO_MGI_slim', '\"disease of subdivision of hemolymphoid system\" EXACT []', ' DOID:7 ! disease of anatomical entity');
INSERT INTO `cirrnainfo_disease` VALUES ('17', 'infertility', 'infertility', ' DOID:5223', '', ' MESH:D007246<> NCI:C3836<> SNOMEDCT_US_2016_03_01:15296000<> SNOMEDCT_US_2016_03_01:162186008<> SNOMEDCT_US_2016_03_01:289179008<> SNOMEDCT_US_2016_03_01:8619003<> UMLS_CUI:C0021359', '', '', '', ' DOID:15 ! reproductive system disease');
INSERT INTO `cirrnainfo_disease` VALUES ('18', 'multiple sclerosis', 'multiple sclerosis', ' DOID:2377', ' \"A demyelinating disease that involves damage to the fatty myelin sheaths around the axons of the brain and spinal cord resulting in demyelination and scarring.\" [url:http://en.wikipedia.org/wiki/Multiple_sclerosis]', ' EFO:0003885<> ICD10CM:G35<> ICD9CM:340<> MESH:D009103<> NCI:C3243<> OMIM:612594<> OMIM:612595<> OMIM:612596<> SNOMEDCT_US_2016_03_01:155023009<> SNOMEDCT_US_2016_03_01:192928003<> SNOMEDCT_US_2016_03_01:192930001<> SNOMEDCT_US_2016_03_01:24700007<> UMLS_CUI:C0026769', '', '', ' \"Generalized multiple sclerosis (disorder)\" EXACT [SNOMEDCT_2005_07_31:192928003]<> \"insular sclerosis\" EXACT [CSP2005:2042-2324]', ' DOID:3213 ! demyelinating disease');
INSERT INTO `cirrnainfo_disease` VALUES ('19', 'bladder cancer', 'urinary bladder cancer', ' DOID:11054', ' \"An urinary system cancer that results_in malignant growth located_in the urinary bladder.\" [url:http://en.wikipedia.org/wiki/Bladder_cancer]', ' ICD10CM:C67<> ICD10CM:C67.9<> ICD9CM:188<> ICD9CM:188.9<> KEGG:05219<> MESH:D001749<> NCI:C2901<> NCI:C9334<> OMIM:109800<> SNOMEDCT_US_2016_03_01:126885006<> SNOMEDCT_US_2016_03_01:154540000<> SNOMEDCT_US_2016_03_01:188248005<> SNOMEDCT_US_2016_03_01:269607003<> SNOMEDCT_US_2016_03_01:363455001<> SNOMEDCT_US_2016_03_01:399326009<> SNOMEDCT_US_2016_03_01:93689003<> UMLS_CUI:C0005684<> UMLS_CUI:C0005695', ' DOID:5428', 'TopNodes_DOcancerslim', '\"bladder cancer\" EXACT []<> \"tumor of the bladder\" EXACT [NCI2004_11_17:C2901]', ' DOID:3996 ! urinary system cancer');
INSERT INTO `cirrnainfo_disease` VALUES ('20', 'liver cancer', 'liver cancer', ' DOID:3571', ' \"A gastrointestinal system cancer that is located_in the liver.\" [url:http://en.wikipedia.org/wiki/Liver]', ' ICD10CM:C22.0<> ICD10CM:C22.9<> ICD9CM:155.0<> ICD9CM:155.2<> MESH:D008113<> NCI:C34803<> NCI:C7692<> SNOMEDCT_US_2016_03_01:126851005<> SNOMEDCT_US_2016_03_01:154470007<> SNOMEDCT_US_2016_03_01:187768001<> SNOMEDCT_US_2016_03_01:187771009<> SNOMEDCT_US_2016_03_01:187779006<> SNOMEDCT_US_2016_03_01:269548006<> SNOMEDCT_US_2016_03_01:363361004<> SNOMEDCT_US_2016_03_01:93870000<> SNOMEDCT_US_2016_03_01:95214007<> UMLS_CUI:C0023903<> UMLS_CUI:C0024620<> UMLS_CUI:C0345904<> UMLS_CUI:C0854795', ' DOID:12300<> DOID:269<> DOID:7330<> DOID:915<> DOID:919', 'TopNodes_DOcancerslim', '\"ca liver - primary\" EXACT []<> \"Ca liver - primary (disorder)\" EXACT [SNOMEDCT_2005_07_31:269548006]<> \"hepatic cancer\" EXACT [CSP2005:2003-2810]<> \"hepatic neoplasm\" EXACT [CSP2005:2003-2810]<>\"malignant hepato-biliary neoplasm\" EXACT []<>\"malignant neoplasm of liver\" EXACT []<> \"malignant neoplasm of liver (disorder)\" EXACT [SNOMEDCT_2005_07_31:93870000]<> \"malignant neoplasm of liver unspecified (disorder)\" EXACT [SNOMEDCT_2005_07_31:187779006]<> \"malignant neoplasm of liver, not specified as primary or secondary\" EXACT [ICD9CM_2006:155.2]<> \"malignant neoplasm of liver, primary\" EXACT [ICD9CM_2006:155.0]<> \"malignant tumor of liver (disorder)\" EXACT [SNOMEDCT_2005_07_31:363361004]<> \"neoplasm of liver (disorder)\" EXACT [SNOMEDCT_2005_07_31:126851005]<>\"non-resectable primary hepatic malignant neoplasm\" EXACT []<>\"primary liver cancer\" EXACT []<> \"primary malignant neoplasm of Liver\" EXACT [NCI2004_11_17:C34803]<> \"primary malignant neoplasm of liver\" EXACT [SNOMEDCT_2005_07_31:187768001, SNOMEDCT_2005_07_31:93870000]<> \"primary malignant neoplasm of liver (disorder)\" EXACT [SNOMEDCT_2005_07_31:95214007]<> \"primary malignant neoplasm of liver NOS (disorder)\" EXACT [SNOMEDCT_2005_07_31:187771009]<> \"Resectable malignant neoplasm of Liver\" EXACT [NCI2004_11_17:C7692]<>\"resectable malignant neoplasm of the liver\" EXACT []', ' DOID:3119 ! gastrointestinal system cancer');
INSERT INTO `cirrnainfo_disease` VALUES ('21', 'breast cancer', 'breast cancer', ' DOID:1612', ' \"A thoracic cancer that originates in the mammary gland.\" [url:http://en.wikipedia.org/wiki/Breast_cancer, url:http://en.wikipedia.org/wiki/Mammary, url:http://www.cancer.gov/cancertopics/types/breast, url:http://www.nlm.nih.gov/medlineplus/breastcancer.html]', ' ICD10CM:C50<> ICD10CM:C50-C50<> ICD9CM:174.8<> MESH:D001943<> NCI:C2910<> NCI:C9335<> OMIM:114480<> OMIM:600048<> OMIM:605365<> SNOMEDCT_US_2016_03_01:126926005<> SNOMEDCT_US_2016_03_01:154521006<> SNOMEDCT_US_2016_03_01:190121004<> SNOMEDCT_US_2016_03_01:254837009<> SNOMEDCT_US_2016_03_01:269595005<> UMLS_CUI:C0006142<> UMLS_CUI:C0153555<> UMLS_CUI:C1458155', ' DOID:1648<> DOID:4241', 'DO_cancer_slim<>TopNodes_DOcancerslim', ' \"breast tumor\" EXACT [NCI2004_11_17:C2910]<>\"malignant neoplasm of breast\" EXACT []<> \"malignant tumor of the breast\" EXACT [NCI2004_11_17:C9335]<>\"mammary cancer\" EXACT []<>\"mammary neoplasm\" RELATED []<> \"mammary tumor\" EXACT [CSP2005:2016-0671]<>\"primary breast cancer\" EXACT []', ' DOID:5093 ! thoracic cancer');
INSERT INTO `cirrnainfo_disease` VALUES ('22', 'prostate cancer', 'prostate cancer', ' DOID:10283', ' \"A male reproductive organ cancer that is located_in the prostate.\" [url:http://www.cancer.gov/dictionary?CdrID=445079]', ' ICD10CM:C61<> ICD9CM:185<> KEGG:05215<> MESH:D011471<> NCI:C3343<> NCI:C7378<> OMIM:176807<> OMIM:300147<> OMIM:300704<> OMIM:601518<> OMIM:602759<> OMIM:608656<> OMIM:608658<> OMIM:609299<> OMIM:609558<> OMIM:610321<> OMIM:610997<> OMIM:611100<> OMIM:611868<> OMIM:611928<> OMIM:611955<> OMIM:611958<> OMIM:611959<> ORDO:1331<> SNOMEDCT_US_2016_03_01:126906006<> SNOMEDCT_US_2016_03_01:363448003<> SNOMEDCT_US_2016_03_01:399068003<> SNOMEDCT_US_2016_03_01:93974005<> UMLS_CUI:C0033578<> UMLS_CUI:C0376358', ' DOID:514', 'DO_FlyBase_slim<>TopNodes_DOcancerslim', '\"hereditary prostate cancer\" EXACT []<> \"malignant tumor of the prostate\" EXACT [NCI2004_11_17:C7378]<> \"NGP - new growth of prostate\" EXACT [SNOMEDCT_2005_07_31:363448003]<> \"prostate cancer, familial\" EXACT [MESH:C537243]<> \"prostate neoplasm\" EXACT [CSP2005:2016-2641]<> \"prostatic cancer\" EXACT [CSP2005:2016-2641]<>\"prostatic neoplasm\" EXACT []<> \"tumor of the prostate\" EXACT [NCI2004_11_17:C3343]', ' DOID:3856 ! male reproductive organ cancer');
INSERT INTO `cirrnainfo_disease` VALUES ('23', 'type 2 diabetes mellitus', 'type 2 diabetes mellitus', ' DOID:9352', ' \"A diabetes mellitus that involves high blood glucose resulting from cells fail to use insulin properly.\" [url:http://en.wikipedia.org/wiki/Diabetes, url:http://en.wikipedia.org/wiki/Diabetes_mellitus_type_2]', ' EFO:0001360<> ICD10CM:E11<> KEGG:04930<> MESH:D003924<> NCI:C26747<> OMIM:125853<> OMIM:601283<> OMIM:601407<> OMIM:603694<> OMIM:608036<> SNOMEDCT_US_2016_03_01:154672006<> SNOMEDCT_US_2016_03_01:190323008<> SNOMEDCT_US_2016_03_01:190384004<> SNOMEDCT_US_2016_03_01:267468009<> SNOMEDCT_US_2016_03_01:44054006<> UMLS_CUI:C0011860', '', 'DO_FlyBase_slim', '\"NIDDM\" EXACT []<>\"non-insulin-dependent diabetes mellitus\" EXACT []<>\"type II diabetes mellitus\" EXACT []', ' DOID:9351 ! diabetes mellitus');
INSERT INTO `cirrnainfo_disease` VALUES ('24', 'stroke', 'cerebrovascular disease', ' DOID:6713', ' \"An artery disease that is characterized by dysfunction of the blood vessels supplying the brain.\" [url:http://en.wikipedia.org/wiki/Cerebrovascular_disease, url:http://www.ncbi.nlm.nih.gov/books/NBK378/]', ' EFO:0000712<> ICD10CM:I60-I69<> ICD10CM:I63.9<> ICD10CM:I67.9<> ICD9CM:430-438.99<> ICD9CM:437.9<> MESH:D002561<> MESH:D020521<> NCI:C2938<> NCI:C3390<> OMIM:601367<> SNOMEDCT_US_2016_03_01:155388006<> SNOMEDCT_US_2016_03_01:155405006<> SNOMEDCT_US_2016_03_01:155412002<> SNOMEDCT_US_2016_03_01:195208004<> SNOMEDCT_US_2016_03_01:195249004<> SNOMEDCT_US_2016_03_01:195595007<> SNOMEDCT_US_2016_03_01:230690007<> SNOMEDCT_US_2016_03_01:266312006<> SNOMEDCT_US_2016_03_01:266315008<> SNOMEDCT_US_2016_03_01:270883006<> SNOMEDCT_US_2016_03_01:313267000<> SNOMEDCT_US_2016_03_01:62914000<> SNOMEDCT_US_2016_03_01:82797006<> UMLS_CUI:C0007820<> UMLS_CUI:C0038454', ' DOID:12214<> DOID:3455<> DOID:8231', '', '\"cerebrovascular accident\" EXACT []<> \"Cerebrovascular accident (disorder)\" EXACT [SNOMEDCT_2005_07_31:230690007]<>\"cerebrovascular disorder\" EXACT []<> \"CVA (cerebral vascular accident)\" EXACT [CSP2005:0617-5539]<>\"stroke\" EXACT []', ' DOID:0050828 ! artery disease');
INSERT INTO `cirrnainfo_disease` VALUES ('25', 'pulmonary tuberculosis', 'pulmonary tuberculosis', ' DOID:2957', ' \"A tuberculosis that is a contagious disease located_in lungs. The infection has_symptom fever, has_symptom cough, has_symptom difficulty in breathing, has_symptom inflammatory infiltrations, has_symptom formation of tubercles, has_symptom caseation, has_symptom pleural effusion, and has_symptom fibrosis.\" [url:http://www.cdc.gov/tb/publications/factsheets/general/tb.htm, url:http://www2.merriam-webster.com/cgi-bin/mwmednlm?book=Medical&va=tuberculosis]', ' ICD10CM:A15<> ICD10CM:A15.0<> ICD9CM:011<> ICD9CM:011.9<> ICD9CM:011.90<> MESH:D014397<> NCI:C26899<> SNOMEDCT_US_2016_03_01:154283005<> SNOMEDCT_US_2016_03_01:186180006<> SNOMEDCT_US_2016_03_01:397190009<> SNOMEDCT_US_2016_03_01:700272008<> SNOMEDCT_US_2016_03_01:81483001<> UMLS_CUI:C0041327', '', 'gram-positive_bacterial_infectious_disease', '', ' DOID:399 ! tuberculosis');
INSERT INTO `cirrnainfo_disease` VALUES ('26', 'brain cancer', 'brain cancer', ' DOID:1319', ' \"A central nervous system cancer that is characterized by the growth of abnormal cells in the tissues of the brain.\" [url:http://www.cancer.gov/dictionary?CdrID=387264]', ' CSP:2006-2736<> ICD10CM:C71<> ICD10CM:C71.9<> ICD9CM:191<> ICD9CM:191.9<> ICD9CM:239.6<> MESH:D001932<> NCI:C2907<> NCI:C3568<> NCI:C4952<> NCI:C4954<> NCI:C5115<> NCI:C7710<> SNOMEDCT_US_2016_03_01:126952004<> SNOMEDCT_US_2016_03_01:154550004<> SNOMEDCT_US_2016_03_01:188279009<> SNOMEDCT_US_2016_03_01:188305001<> SNOMEDCT_US_2016_03_01:189537005<> SNOMEDCT_US_2016_03_01:254935002<> SNOMEDCT_US_2016_03_01:269610005<> SNOMEDCT_US_2016_03_01:275464006<> SNOMEDCT_US_2016_03_01:363356001<> SNOMEDCT_US_2016_03_01:428061005<> SNOMEDCT_US_2016_03_01:93727008<> UMLS_CUI:C0006118<> UMLS_CUI:C0153633<> UMLS_CUI:C0220624<> UMLS_CUI:C0750974<> UMLS_CUI:C0750979<> UMLS_CUI:C1334557', ' DOID:2125<> DOID:2126<> DOID:3543<> DOID:6649<> DOID:911', 'DO_cancer_slim<>DO_FlyBase_slim<>TopNodes_DOcancerslim', '\"adult brain tumor\" EXACT []<>\"adult malignant brain neoplasm\" EXACT []<> \"brain neoplasm\" EXACT [CSP2005:2006-2736]<> \"brain neoplasm, adult\" EXACT [NCI2004_11_17:C7710]<> \"BT - Brain tumour\" EXACT [SNOMEDCT_2005_07_31:254935002]<> \"malignant brain tumour\" EXACT [SNOMEDCT_2005_07_31:363356001]<> \"malignant primary brain neoplasm\" EXACT [NCI2004_11_17:C4954]<>\"malignant primary brain tumor\" EXACT []<> \"malignant tumor of adult brain\" EXACT [NCI2004_11_17:C5115]<> \"malignant tumor of Brain\" EXACT [NCI2004_11_17:C3568]<> \"neoplasm of brain (disorder)\" EXACT [SNOMEDCT_2005_07_31:126952004]<> \"neoplasm of unspecified nature of brain\" EXACT [ICD9CM_2006:239.6]<> \"neoplasm of unspecified nature of brain (disorder)\" EXACT [SNOMEDCT_2005_07_31:189537005]<> \"primary brain neoplasm\" EXACT [NCI2004_11_17:C4952]<>\"primary brain tumor\" EXACT []<> \"primary malignant neoplasm of brain (disorder)\" EXACT [SNOMEDCT_2005_07_31:93727008]<> \"tumor of the Brain\" EXACT [NCI2004_11_17:C2907]', ' DOID:3620 ! central nervous system cancer');
INSERT INTO `cirrnainfo_disease` VALUES ('27', 'atherosclerosis', 'atherosclerosis', ' DOID:1936', '', ' EFO:0003914<> ICD10CM:I25.1<> ICD10CM:I70<> ICD9CM:440<> MESH:D050197<> NCI:C35768<> NCI:C35771<> SNOMEDCT_US_2016_03_01:155382007<> SNOMEDCT_US_2016_03_01:155414001<> SNOMEDCT_US_2016_03_01:194848007<> SNOMEDCT_US_2016_03_01:195251000<> SNOMEDCT_US_2016_03_01:266318005<> SNOMEDCT_US_2016_03_01:38716007<> UMLS_CUI:C0004153', '', '', '', ' DOID:2348 ! arteriosclerotic cardiovascular disease');

-- ----------------------------
-- Table structure for cirrnainfo_relationship
-- ----------------------------
DROP TABLE IF EXISTS `cirrnainfo_relationship`;
CREATE TABLE `cirrnainfo_relationship` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `cir_bace_id` varchar(100) DEFAULT NULL,
  `cir_id` varchar(500) DEFAULT NULL,
  `disease_name` varchar(500) DEFAULT NULL,
  `pmid` varchar(500) DEFAULT NULL,
  `evidence` varchar(5000) DEFAULT NULL,
  `displaycir_id` varchar(500) DEFAULT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=119 DEFAULT CHARSET=utf8;

-- ----------------------------
-- Records of cirrnainfo_relationship
-- ----------------------------
INSERT INTO `cirrnainfo_relationship` VALUES ('1', 'hsa_circ_002059', '', 'gastric carcinoma', 'PMID: 25689795', 'Hsa_circ_002059, a typical circular RNA, was first found to be significantly downregulated in gastric cancer tissues compared with paired adjacent nontumorous tissues (p<0.001).', 'hsa_circ_002059');
INSERT INTO `cirrnainfo_relationship` VALUES ('2', '', 'cir-ITCH', 'esophageal squamous cell carcinoma', 'PMID: 25749389', 'These results indicate that cir-ITCH may have an inhibitory effect on ESCC by regulating the Wnt pathway.', 'cir-ITCH');
INSERT INTO `cirrnainfo_relationship` VALUES ('3', '', 'hsa_circRNA_100783', 'ageing', 'PMID: 26451160 ', 'The overlapping expression of circular RNA100783 may represent a novel biomarker for the longitudinal tracking ofCD28-related CD8(+) T cell ageing and global immunosenescence.', 'hsa_circRNA_100783');
INSERT INTO `cirrnainfo_relationship` VALUES ('4', 'Hsa_circ_0001649', '', 'hepatocellular carcinoma', 'PMID: 26600397', 'hsa_circ_0001649 expression was significantly downregulated in HCC tissues (p = 0.0014) based on an analysis of 89 paired samples of HCC and adjacent liver tissues and the area under the ROC curve (AUC) was 0.63.', 'Hsa_circ_0001649');
INSERT INTO `cirrnainfo_relationship` VALUES ('5', '', 'hsa_circ_101222', 'pre-eclampsia', 'PMID: 26846540', 'The circ_101222 levels in blood corpuscles of patients with PE were significantly higher than those in corresponding healthy women (P<0.001).', 'hsa_circ_101222');
INSERT INTO `cirrnainfo_relationship` VALUES ('6', '', 'circ-Foxo3', 'cardiac senescence', 'PMID: 26873092 ', 'We conclude that ID-1, E2F1, FAK, and HIF1 interact with circ-Foxo3 and are retained in the cytoplasm and could no longer exert their anti-senescent and anti-stress roles, resulting in increased cellular senescence.', 'circ-Foxo3');
INSERT INTO `cirrnainfo_relationship` VALUES ('7', '', 'hsa_circ_001988', 'colorectal cancer ', 'PMID: 26884878', 'The expression of hsa_circ_001988 in 31 matched colorectal cancer tissue and normal colon mucosa was analyzed by quantitative real-time polymerase chain reaction (qRT-PCR). We used Ct method and investigated the differences between tumor tissues and normal colon mucosa by paired t-test. One-way analysis of variance was conducted to analyze the relationship between hsa_circ_001988 expression level and clinic pathological factors of patients. Receiver operating characteristic (ROC) curve was built by SPSS to evaluate the diagnostic values. The expression of hsa_circ_001988 was significantly correlated with differentiation (P<0.05) and perineural invasion (P<0.05). The area under ROC curve of hsa_circ_001988 was 0.788 (P<0.05).', 'hsa_circ_001988');
INSERT INTO `cirrnainfo_relationship` VALUES ('8', '', 'circRNA-CER/circRNA_100876', 'osteoarthritis', 'PMID: 26931159', 'CircRNA-CER expression increased with interleukin-1 and tumor necrosis factor levels in chondrocytes. Silencing of circRNA-CER using small interfering RNA suppressed MMP13 expression and increased ECM formation. CircRNA-CER could compete for miR-136 with MMP13. Our results demonstrated that circRNA-CER regulated MMP13 expression by functioning as a competing endogenous RNA (ceRNA) and participated in the process of chondrocyte ECM degradation. We propose that circRNA-CER could be used as a potential target in OA therapy.', 'circRNA-CER/circRNA_100876');
INSERT INTO `cirrnainfo_relationship` VALUES ('9', '', 'Cdr1as/CiRS-7', 'Myocardial Infarction', 'PMID: 26998750', 'Cdr1as also functioned as a powerful miR-7a sponge in myocardial cells, and showed regulation on the protective role of miR-7a in MI injury, involving the function of miR-7a targets, PARP and SP1.', 'Cdr1as/CiRS-7');
INSERT INTO `cirrnainfo_relationship` VALUES ('10', '', 'hsa_circRNA_100855', 'laryngeal squamous cell cancer', 'PMID: 27158380 ', 'Results showed that hsa_circRNA_100855 level was significantly higher in LSCC than in the corresponding adjacent non-neoplastic tissues. Patients with T3-4 stage, neck nodal metastasis or advanced clinical stage had higher hsa_circRNA_100855 expression.', 'hsa_circRNA_100855');
INSERT INTO `cirrnainfo_relationship` VALUES ('11', '', 'hsa_circRNA_104912', 'laryngeal squamous cell cancer', 'PMID: 27158380 ', 'The hsa_circRNA_104912 level was significantly lower in LSCC than in corresponding adjacent non-neoplastic tissues. Patients with T3-4 stage, neck nodal metastasis, poor differentiation or advanced clinical stage had a lower hsa_circRNA_104912 expression.', 'hsa_circRNA_104912');
INSERT INTO `cirrnainfo_relationship` VALUES ('12', 'hsa_circ_0005075', '', 'hepatocellular carcinoma', 'PMID: 27258521', 'Only hsa_circ_0005075 exhibited significant difference in expression (P<0.001) between HCC and normal tissues. Hsa_circ_0005075 expression correlated with HCC tumor size (P=0.042), and showed good diagnostic potential (AUROC=0.94). Finally, we constructed a network of hsa_circ_0005075-targeted miRNA-gene interactions, including miR-23b-5p, miR-93-3p, miR-581, miR-23a-5p, and their corresponding mRNAs. Gene oncology analysis revealed that hsa_circ_0005075 could participate in cell adhesion during HCC development. In summary, we identified hsa_circ_0005075 as a potential HCC biomarker; however, further studies are required to confirm the role of this circRNA, and others, in HCC development.', 'hsa_circ_0005075');
INSERT INTO `cirrnainfo_relationship` VALUES ('13', '', 'hsa_circRNA_103636', 'major depressive disorder', 'PMID: 27404501', 'Four differentially expressed circRNAs were identified between MDD patients and controls, and only down-regulated hsa_circRNA_103636 was significantly altered after the 8-week treatment in MDD patients.', 'hsa_circRNA_103636');
INSERT INTO `cirrnainfo_relationship` VALUES ('14', '', 'circTCF25', 'bladder carcinoma', 'PMID: 27484176', 'We further predicted that circTCF25 could sequester miR-103a-3p/miR-107, which potentially lead to the up-regulation of thirteen targets related to cell proliferation, migration and invasion. Subsequently, we demonstrated that over-expression of circTCF25 could down-regulate miR-103a-3p and miR-107, increase CDK6 expression, and promote proliferation and migration in vitro and vivo. This is the first study to exploit circRNA profiling and circRNA/miRNA interactions in bladder cancer. Our work laid the foundation to investigate the functions of circRNAs in cancers. The data also suggest that circTCF25 might be a new promising marker for bladder cancer.', 'circTCF25');
INSERT INTO `cirrnainfo_relationship` VALUES ('15', '', 'hsa_circRNA_100782', 'preeclampsia', 'PMID: 27606420', 'The expression levels of hsa_circRNA_100782 (p < 0.05), hsa_circRNA_102682 (p < 0.05), and hsa_circRNA_104820 (p < 0.0001) were validated as significantly up-regulated in the experimental group compared with the controls.', 'hsa_circRNA_100782');
INSERT INTO `cirrnainfo_relationship` VALUES ('16', '', 'hsa_circRNA_102682', 'preeclampsia', 'PMID: 27606420', 'The expression levels of hsa_circRNA_100782 (p < 0.05), hsa_circRNA_102682 (p < 0.05), and hsa_circRNA_104820 (p < 0.0001) were validated as significantly up-regulated in the experimental group compared with the controls.', 'hsa_circRNA_102682');
INSERT INTO `cirrnainfo_relationship` VALUES ('17', '', 'hsa_circRNA_104820', 'preeclampsia', 'PMID: 27606420', 'The expression levels of hsa_circRNA_100782 (p < 0.05), hsa_circRNA_102682 (p < 0.05), and hsa_circRNA_104820 (p < 0.0001) were validated as significantly up-regulated in the experimental group compared with the controls.', 'hsa_circRNA_104820');
INSERT INTO `cirrnainfo_relationship` VALUES ('18', '', 'cZNF292', 'glioma', 'PMID: 27613831 ', 'We found that cZNF292 silencing suppresses tube formation by inhibiting glioma cell proliferation and cell cycle progression. Cell cycle progression in human glioma U87MG and U251 cells was halted at S/G2/M phase via the Wnt/-catenin signaling pathway and related genes such as PRR11, Cyclin A, p-CDK2, VEGFR-1/2, p-VEGFR-1/2 and EGFR. The results suggest that cZNF292 silencing plays an important role in the tube formation process and has potential for application as a therapeutic target and biomarker in glioma.', 'cZNF292');
INSERT INTO `cirrnainfo_relationship` VALUES ('19', '', 'ciRS-7/CDR1-AS', 'hepatocellular carcinoma', 'PMID: 27614453', 'The expression levels of ciRS-7 were comparable between HCC and matched non-tumor tissues. However, the highly ciRS-7 expression in HCC tissues was significantly correlated with hepatic MVI, AFP level and younger age and thus partly related with the deterioration of HCC. Especially, ciRS-7 was one of the independent factors of hepatic MVI. These data suggested that ciRS-7 may be a promising biomarker of hepatic MVI and a novel therapy target for restraining MVI.', 'ciRS-7/CDR1-AS');
INSERT INTO `cirrnainfo_relationship` VALUES ('20', '', 'cir-ITCH', 'lung cancer', 'PMID: 27642589 ', 'Ectopic expression of cir-ITCH markedly elevated its parental cancer-suppressive gene, ITCH, expression and inhibited proliferation of lung cancer cells. Molecular analysis further revealed that cir-ITCH acted as sponge of oncogenic miR-7 and miR-214 to enhance ITCH expression and thus suppressed the activation of Wnt/-catenin signaling.', 'cir-ITCH');
INSERT INTO `cirrnainfo_relationship` VALUES ('21', 'hsa_circ_0067934', '', 'Esophageal squamous cell carcinoma', 'PMID: 27752108 ', 'The in vitro silence of hsa_circ_0067934 by siRNA inhibited the proliferation and migration of ESCC cells and blocked cell cycle progression. Cell fraction analyses and fluorescence in situ hybridization detected that hsa_circ_0067934 was mostly located in the cytoplasm. Our findings suggest that hsa_circ_0067934 is upregulated in ESCC tumor tissue.', 'hsa_circ_0067934');
INSERT INTO `cirrnainfo_relationship` VALUES ('22', 'hsa_circ_0000615', 'cir-ZNF609', 'Hirschsprung disease', 'PMID: 27903978 ', 'We reported that the cir-ZNF609 was down-regulated in HSCR compared with normal bowel tissues. Furthermore, suppression of cir-ZNF609 inhibited the proliferation and migration of cells. We screened out several putative cir-ZNF609 ceRNAs of which the AKT3 transcript was selected. Finally, RNA immunoprecipitation and luciferase reporter assays demonstrated that cir-ZNF609 may act as a sponge for miR-150-5p to modulate the expression of AKT3. In conclusion, these findings illustrated that cir-ZNF609 took part in the onset of HSCR through the crosstalk with AKT3 by competing for shared miR-150-5p.', 'cir-ZNF609');
INSERT INTO `cirrnainfo_relationship` VALUES ('23', 'hsa_circ_0000069', '', 'colorectal cancer', 'PMID: 28003761 ', 'A high expression of hsa_circ_0000069 was observed in CRC tissues and correlated with patients^ age and tumor, node, metastasis (TNM) stage (P<0.05). Furthermore, by using specifically designed siRNAs in CRC cells, a functional analysis was performed which revealed that hsa_circ_0000069 knockdown could notably inhibit cell proliferation, migration, and invasion, and induce G0/G1 phase arrest of cell cycle in vitro.', 'hsa_circ_0000069');
INSERT INTO `cirrnainfo_relationship` VALUES ('24', 'hsa_circ_0124644', '', 'coronary artery disease', 'PMID: 28045102 ', 'Hsa_circ_0124644 had the largest area under the curve (AUC). We tested hsa_circ_0124644 in an independent cohort consisting of 115 control individuals and 137 CAD patients. After we included the risk factors for CAD, the AUC slightly increased from 0.769 (95% confidence interval=[0.710-0.827], P<0.001) to 0.804 ([0.751-0.857], P<0.001), and when combined with hsa_circ_0098964, the diagnostic value slightly increased. Taken together, our results suggest that hsa_circ_0124644 can be used as a diagnostic biomarker of CAD.', 'hsa_circ_0124644');
INSERT INTO `cirrnainfo_relationship` VALUES ('25', 'Hsa_circ_0000096', '', 'gastric cancer', 'PMID: 28081541', 'Hsa_circ_0000096 was found to be significantly downregulated in gastric cancer tissues and gastric cancer cell lines compared with paired adjacent non-tumorous tissues and normal gastric epithelial cells (P<0.001). Moreover, knockdown of hsa_circ_0000096 significantly inhibited cell proliferation and migration in vitro and in vivo. The results of both immunohistochemical and western blot analyses showed that the protein levels of cyclin D1, cyclin-dependent kinase 6 (CDK6), matrix metalloproteinase-2 and MMP-9 were significantly reduced in vitro and in vivo. A gastric cancer xenograft nude mouse model indicated that Ki67 and VEGF were reduced in a dose-dependent manner following knockdown of hsa_circ_0000096. However, the expression of E-cadherin increased.', 'Hsa_circ_0000096');
INSERT INTO `cirrnainfo_relationship` VALUES ('26', '', 'circSLC8A1-1', 'cardiac-expressed', 'PMID: 28082450', 'The most abundant cardiac-expressed circRNA is a cytoplasmic localized single-exon circSLC8A1-1. The longest human transcript TTN alone generates up to 415 different exonic circRNA isoforms, the majority (83%) of which originates from the I-band domain. Finally, we confirmed the expression of selected cardiac circRNA by RT-PCR, Sanger sequencing and single molecule RNA-fluorescence in situ hybridization.', 'circSLC8A1-1');
INSERT INTO `cirrnainfo_relationship` VALUES ('27', '', 'BANP', 'colorectal cancer ', 'PMID: 28103507', 'We further validated one circRNA generated from Exon 5-11 of BANP gene, termed circ-BANP. In addition, RT-PCR result showed that circ-BANP was overexpressed in 35 CRC cancerous tissues. Knockdown of circ-BANP with siRNA significantly attenuate the proliferation of CRC cells. In summary, our findings demonstrated that dysregulated circ-BANP appears to have an important role in CRC cells and could serve as a prognostic and therapeutic marker for CRC.', 'BANP');
INSERT INTO `cirrnainfo_relationship` VALUES ('28', 'hsa_circ_0000190', '', 'gastric cancer', 'PMID: 28130019', 'Hsa_circ_0000190 was first found to be down-regulated in gastric cancer tissues (P<0.001) and plasma samples from patients with gastric cancer (P<0.001). Its expression levels were significantly correlated with tumor diameter (P=0.034), lymphatic metastasis (P=0.026), distal metastasis (P=0.001), TNM stage (P=0.001), and CA19-9 levels (P=0.019). The areas under the ROC curve (AUC) of hsa_circ_0000190 in tissues and plasma were up to 0.75 and 0.60, respectively. The sensitivity and specificity of the combination were 0.712 and 0.750; the AUC was increased to 0.775.', 'hsa_circ_0000190');
INSERT INTO `cirrnainfo_relationship` VALUES ('29', '', 'ciRS-7/CDR1-AS', 'Colorectal Cancer', 'PMID: 28174233 ', 'Our data revealed that ciRS-7 was significantly upregulated in colorectal cancer tissues compared with matched normal mucosae (P = 0.0018), and its overexpression was associated with poor patient survival (P = 0.0224 and 0.0061 in the training and validation cohorts, respectively). Multivariate survival analysis revealed that ciRS-7 emerged as an independent risk factor for overall survival (P = 0.0656 and 0.0324 in the training and validation cohorts, respectively). Overexpression of ciRS-7 in HCT116 and HT29 cells led to the blocking of miR-7 and resulted in a more aggressive oncogenic phenotype, and ciRS-7 overexpression permitted the inhibition of miR-7 and subsequent activation of EGFR and RAF1 oncogenes.', 'ciRS-7/CDR1-AS');
INSERT INTO `cirrnainfo_relationship` VALUES ('30', '', 'hsa_circRNA_000543', 'gastric cancer', 'PMID: 28184940 ', 'The qRT-PCR validation experiments showed that hsa_circRNA_400071, hsa_circRNA_000543 and hsa_circRNA_001959 expression was consistent with the microarray analysis results.', 'hsa_circRNA_000543');
INSERT INTO `cirrnainfo_relationship` VALUES ('31', '', 'hsa_circRNA_001959', 'gastric cancer', 'PMID: 28184940 ', 'The qRT-PCR validation experiments showed that hsa_circRNA_400071, hsa_circRNA_000543 and hsa_circRNA_001959 expression was consistent with the microarray analysis results.', 'hsa_circRNA_001959');
INSERT INTO `cirrnainfo_relationship` VALUES ('32', '', 'hsa_circRNA_400071', 'gastric cancer', 'PMID: 28184940 ', 'The qRT-PCR validation experiments showed that hsa_circRNA_400071, hsa_circRNA_000543 and hsa_circRNA_001959 expression was consistent with the microarray analysis results.', 'hsa_circRNA_400071');
INSERT INTO `cirrnainfo_relationship` VALUES ('33', '', 'cirZKSCAN1', 'hepatocellular carcinoma', 'PMID: 28211215 ', 'Here, we analyzed a cohort of 102 patients and found that expression of both ZKSCAN1mRNA and circZKSCAN1 was significantly lower (P<0.05) in the HCC samples compared with that in matched adjacent nontumorous tissues by reverse transcription PCR (RT-PCR). The low expression level of ZKSCAN1 was only associated with tumor size (P=0.032), while the cirZKSCAN1 levels varied in patients with different tumor numbers (P<0.01), cirrhosis (P=0.031), vascular invasion (P=0.002), or microscopic vascular invasion (P=0.002), as well as with the tumor grade (P<0.001).', 'cirZKSCAN1');
INSERT INTO `cirrnainfo_relationship` VALUES ('34', 'hsa_circ_0000594', 'circ-TTBK2', 'glioma malignancy', 'PMID: 28219405 ', 'We found that circ-TTBK2 was upregulated in glioma tissues and cell lines, while linear TTBK2 was not dysregulated in glioma tissues and cells. Enhanced expression of circ-TTBK2 promoted cell proliferation, migration, and invasion, while inhibited apoptosis. MiR-217 was downregulated in glioma tissues and cell lines. We also found that circ-TTBK2, but not linear TTBK2, acted as miR-217 sponge in a sequence-specific manner. In addition, upregulated circ-TTBK2 decreased miR-217 expression and there was a reciprocal negative feedback between them in an Argonaute2-dependent manner. Moreover, reintroduction of miR-217 significantly reversed circ-TTBK2-mediated promotion of glioma progression. HNF1 was a direct target of miR-217, and played oncogenic role in glioma cells. Remarkably, circ-TTBK2 knockdown combined with miR-217 overexpression led to tumor regression in vivo.', 'circ-TTBK2');
INSERT INTO `cirrnainfo_relationship` VALUES ('35', '', 'Circ-ZNF609', 'Duchenne muscular dystrophy', 'PMID: 28344082', 'One of them, circ-ZNF609, resulted in specifically controlling myoblast proliferation. Circ-ZNF609 contains an open reading frame spanning from the start codon, in common with the linear transcript, and terminating at an in-frame STOP codon, created upon circularization. Circ-ZNF609 is associated with heavy polysomes, and it is translated into a protein in a splicing-dependent and cap-independent manner, providing an example of a protein-coding circRNA in eukaryotes.', 'Circ-ZNF609');
INSERT INTO `cirrnainfo_relationship` VALUES ('36', '', 'hsa_circRNA_103809', 'colorectal cancer', 'PMID: 28349836', 'We also first demonstrated that the expression levels of hsa_circRNA_103809 ( p<0.0001) and hsa_circRNA_104700 ( p=0.0003) were significantly lower in colorectal cancer than in normal tissues. The expression level of hsa_circRNA_103809 was significantly correlated with lymph node metastasis ( p=0.021) and tumor-node-metastasis stage ( p=0.011), and the expression level of hsa_circRNA_104700 was significantly correlated with distal metastasis ( p=0.036). The area under receiver operating characteristic curves of hsa_circRNA_103809 and hsa_circRNA_104700 were 0.699 ( p<0.0001) and 0.616 ( p<0.0001), respectively. In conclusion, these results suggest that hsa_circRNA_103809 and hsa_circRNA_104700 may be potentially involved in the development of colorectal cancer and serve as potential biomarkers for the diagnosis of colorectal cancer.', 'hsa_circRNA_103809');
INSERT INTO `cirrnainfo_relationship` VALUES ('37', '', 'hsa_circRNA_104700', 'colorectal cancer', 'PMID: 28349836', 'We also first demonstrated that the expression levels of hsa_circRNA_103809 ( p<0.0001) and hsa_circRNA_104700 ( p=0.0003) were significantly lower in colorectal cancer than in normal tissues. The expression level of hsa_circRNA_103809 was significantly correlated with lymph node metastasis ( p=0.021) and tumor-node-metastasis stage ( p=0.011), and the expression level of hsa_circRNA_104700 was significantly correlated with distal metastasis ( p=0.036). The area under receiver operating characteristic curves of hsa_circRNA_103809 and hsa_circRNA_104700 were 0.699 ( p<0.0001) and 0.616 ( p<0.0001), respectively. In conclusion, these results suggest that hsa_circRNA_103809 and hsa_circRNA_104700 may be potentially involved in the development of colorectal cancer and serve as potential biomarkers for the diagnosis of colorectal cancer.', 'hsa_circRNA_104700');
INSERT INTO `cirrnainfo_relationship` VALUES ('38', 'hsa_circ_0005986', '', 'hepatocellular carcinoma', 'PMID: 28410211 ', 'We revealed that the expression level of hsa_circ_0005986 in HCC was significantly lower than that in adjacent non-tumorous tissues (P < 0.001). Its levels in HCC cell lines, HepG2, SMMC7721, Huh7, MHCC97L, MHCC97H, and HCCLM3 were significantly lower than those in human normal hepatic cell line L02 (P < 0.001). In addition, the low expression level of hsa_circ_0005986 was correlated with chronic hepatitis B family history (P = 0.001), tumor diameters (P < 0.001), microvascular invasion (P = 0.026), and Barcelona Clinic Liver Cancer (BCLC) stage (P < 0.001). Further experiments demonstrated that both hsa_circ_0005986 and Notch1mRNA were targets of miR-129-5p, and that hsa_circ_0005986 downregulation liberated miR-129-5p and decreased the expression level of Notch1mRNA. More importantly, hsa_circ_0005986 downregulation accelerated cell proliferation by promoting the G0/G1 to S phase transition. We conclude that hsa_circ_0005986 function as microRNA sponge in tumorigenesis and can be used as a novel biomarker for HCC.', 'hsa_circ_0005986');
INSERT INTO `cirrnainfo_relationship` VALUES ('39', '', 'circRNA_010567', 'cardiac fibroblasts', 'PMID: 28412345', 'Furthermore, circRNA_010567 was markedly up-regulated in diabetic mice myocardium and cardiac fibroblasts (CFs) treated with Ang II. Bioinformatics analysis predicted circRNA_010567, sponge miR-141 and miR-141 directly target TGF-1, which was validated by dual-luciferase assay. Subsequently, functional experiments revealed circRNA_010567 silencing could up-regulate miR-141 and down-regulate TGF-1 expression, and suppress fibrosis-associated protein resection in CFs, including Col I, Col III and -SMA. Results demonstrate the circRNA_010567/miR-141/TGF-1 axis plays an important regulatory role in the diabetic mice myocardial fibrosis model. The present study characterizes a new function of circRNA in the pathogenesis of myocardial fibrosis in a diabetic mouse model, providing novel insight for circRNA-miRNA-mRNA in cardiovascular disease.', 'circRNA_010567');
INSERT INTO `cirrnainfo_relationship` VALUES ('40', 'hsa-circ-0016347', '', 'osteosarcoma', 'PMID: 28424426', 'We demonstrated that circ-0016347 acted as a positive regulator in osteosarcoma cells proliferation and invasion. Moreover, circ-0016347 was identified as a sponge of miR-214 that upregulated the expression of caspase-1, which is the functional target of miR-214. Our study provides novel insight into the regulatory mechanism of circ-0016347 and its downstream targets in proliferation, invasion and metastasis of osteosarcoma cells, which will facilitate further development in the therapy of osteosarcoma.', 'hsa-circ-0016347');
INSERT INTO `cirrnainfo_relationship` VALUES ('41', 'hsa_circ_0001895', '', 'gastric cancer', 'PMID: 28443463 ', 'We chose hsa_circ_0001895 as a targeted circRNA to investigate its clinical significances in gastric cancer patients. Hsa_circ_0001895 expression levels in five gastric cancer cell lines and 257 specimens of tissues were measured by real-time quantitative reverse transcription polymerase chain reaction. Then, the potential relationship between hsa_circ_0001895 expression levels and patients^ clinicopathological factors was investigated. A receiver operating characteristic curve was constructed for evaluating the diagnostic value of hsa_circ_0001895. Hsa_circ_0001895 expression levels in five detected gastric cancer cell lines (AGS, BGC-823, HGC-27, MGC-803, and SGC-7901) were all significantly downregulated than those in normal gastric epithelial GES-1 cells. Besides, compared with healthy control tissues, it was downregulated not only in 69.8% (67/96) gastric cancer tissues but also in gastric precancerous lesions. Moreover, hsa_circ_0001895 expression levels were significantly correlated with cell differentiation, Borrmann type, and tissue carcino-embryonic antigen expression. Our results suggested that hsa_circ_0001895 may play crucial roles during gastric cancerogenesis and is a potential biomarker for clinical prognosis prediction.', 'hsa_circ_0001895');
INSERT INTO `cirrnainfo_relationship` VALUES ('42', 'hsa_circ_0003570', '', 'hepatocellular carcinoma', 'PMID: 28493512 ', 'Hsa_circ_0003570 was not only first found down-regulated in HCC cell lines (P<.001) but also in HCC tissues (P<.001). Moreover, hsa_circ_0003570 was gradually decreased from chronic hepatitis (CH), to liver cirrhosis (LC) and to HCC tissues (P<.01). Its expression levels were significantly correlated with tumor diameter (P=.035), differentiation (P=.013), microvascular invasion (P=.045), Barcelona Clinic Liver Cancer stages (P=.011), tumor-node-metastasis stages (P=.016), and serum alpha-fetoprotein levels (P=.031). The ROC curve demonstrated that hsa_circ_0003570 had poor performance for differentiating HCC from LC and CH, but had relatively good performance for differentiating LC from CH.', 'hsa_circ_0003570');
INSERT INTO `cirrnainfo_relationship` VALUES ('43', '', 'MFACR', 'myocardial infarction', 'PMID: 28498369 ', 'We show that a circRNA (mitochondrial fission and apoptosis-related circRNA (MFACR)) regulates mitochondrial fission and apoptosis in the heart by directly targeting and downregulating miR-652-3p; this in turn blocks mitochondrial fission and cardiomyocyte cell death by suppressing MTP18 translation. MTP18 deficiency reduces mitochondrial fission and suppresses cardiomyocyte apoptosis and MI. miR-652-3p directly downregulates MTP18 and attenuates mitochondrial fission, cardiomyocyte apoptosis, and MI in vitro and in vivo. MFACR directly sequesters miR-652-3p in the cytoplasm and inhibits its activity. MFACR knockdown in cardiomyocytes and mice attenuates mitochondrial fission and MI. Our results reveal a crucial role for circRNA in regulating mitochondrial dynamics and apoptosis in the heart; as such, circRNAs may serve as a potential therapeutic avenue for cardiovascular diseases.', 'MFACR');
INSERT INTO `cirrnainfo_relationship` VALUES ('44', '', 'hsa_circRNA_0007874', 'hepatocellular carcinoma', 'PMID: 28520103', 'We analyzed the expression profile of human circRNAs in HCC tissues and identified circMTO1 (mitochondrial translation optimization 1 homologue; hsa_circRNA_0007874/hsa_circRNA_104135) as one circRNA significantly down-regulated in HCC tissues. HCC patients with low circMTO1 expression had shortened survival.', 'hsa_circRNA_0007874');
INSERT INTO `cirrnainfo_relationship` VALUES ('45', '', 'hsa_circRNA_104135', 'hepatocellular carcinoma', 'PMID: 28520103', 'We analyzed the expression profile of human circRNAs in HCC tissues and identified circMTO1 (mitochondrial translation optimization 1 homologue; hsa_circRNA_0007874/hsa_circRNA_104135) as one circRNA significantly down-regulated in HCC tissues. HCC patients with low circMTO1 expression had shortened survival.', 'hsa_circRNA_104135');
INSERT INTO `cirrnainfo_relationship` VALUES ('46', 'Hsa_circ_0014717', '', 'gastric cancer', 'PMID: 28544609', 'Hsa_circ_0014717 was significantly downregulated in 77.2% (74/96) gastric cancer tissues. Its levels in gastric cancer tissues were related to tumor stage (P=0.037), distal metastasis (P=0.048), tissue carcinoembryonic antigen (P=0.001), and carbohydrate antigen 19-9 expression (P=0.021). More importantly, hsa_circ_0014717 can stably exist in human gastric juice; and its nature meets the requirements of clinical detection.', 'Hsa_circ_0014717');
INSERT INTO `cirrnainfo_relationship` VALUES ('47', 'hsa_circ_0010729', '', 'cardiovascular system disease', 'PMID: 28571741', 'From the over-expressed circRNAs, hsa_circ_0010729 was selected as candidate circRNA and which was validated to be significantly up-regulated using RT-PCR. In loss-of-function experiments of HUVECs, hsa_circ_0010729 knockdown suppressed the proliferation and migration ability and enhanced apoptosis. Bioinformatic prediction and luciferase assay revealed that hsa_circ_0010729 and hypoxia inducible factor 1 alpha (HIF-1) were targeted by miR-186. Validation experiments verified that hsa_circ_0010729 was co-expressed with HIF-1, being negatively correlated with miR-186. Moreover, rescue experiments demonstrated that miR-186 inhibitor could reverse the role of hsa_circ_0010729 knockdown on HUVECs progression. Overall, the present study identifies the crucial regulation of hsa_circ_0010729 on vascular endothelial cell proliferation and apoptosis via targeting miR-186/HIF-1 axis.', 'hsa_circ_0010729');
INSERT INTO `cirrnainfo_relationship` VALUES ('48', 'hsa_circ_0003159', '', 'gastric cancer', 'PMID: 28618205 ', 'Compared with paired adjacent non-tumorous tissues, hsa_circ_0003159 expression was significantly down-regulated in gastric cancer tissues. What is more, we found that hsa_circ_0003159 expression levels were significantly negatively associated with gender, distal metastasis, and tumor-node-metastasis stage.', 'hsa_circ_0003159');
INSERT INTO `cirrnainfo_relationship` VALUES ('49', '', 'circ-ITCH', 'Hepatocellular carcinoma', 'PMID: 28636993 ', 'We found rs10485505 (adjusted OR =1.18; 95% CI=1.06-1.31; P value =3.110-3) and rs4911154 (adjusted OR =1.27; 95% CI=1.14-1.43; P value =3.710-5) were significantly associated with increased HCC risk. The expression level of circ-ITCH was significantly lower in HCC tissues, compared with that in adjacent tissues (P value < 0.001). Cox regression analysis indicated that high expression of circ-ITCH was associated with favorable survival of HCC (HR=0.45; 95% CI=0.29-0.68; P value < 0.001). These results indicate that circ-ITCH may have an inhibitory effect on HCC, and could serve as susceptibility and prognostic biomarkers for HCC patients.', 'circ-ITCH');
INSERT INTO `cirrnainfo_relationship` VALUES ('50', 'hsa_circ_0058246', '', 'Gastric cancer', 'PMID: 28639908', 'In particular, we found that the expression of hsa_circ_0058246 was elevated in tumor specimens of patients with poor clinical outcomes. Our collective findings indicate that circular RNAs play a critical role in gastric cancer tumorigenesis. Data from this study provide a new perspective on the molecular pathways underlying metastasis and recurrence of gastric cancer and highlight potential therapeutic targets that may contribute to more effective diagnosis and treatment of the disease.', 'hsa_circ_0058246');
INSERT INTO `cirrnainfo_relationship` VALUES ('51', '', 'circRNA_103827', 'infertility', 'PMID: 28644873', 'Bioinformatics analysis revealed that both circRNAs were potentially involved in glucose metabolism, mitotic cell cycle, and ovarian steroidogenesis. Therefore, age-related up-regulation of circRNA_103827 and circRNA_104816 might be potential indicators of compromised follicular micro-environment which could be used to predict IVF prognosis, and improve female infertility management.', 'circRNA_103827');
INSERT INTO `cirrnainfo_relationship` VALUES ('52', '', 'circRNA_104816', 'infertility', 'PMID: 28644873', 'Bioinformatics analysis revealed that both circRNAs were potentially involved in glucose metabolism, mitotic cell cycle, and ovarian steroidogenesis. Therefore, age-related up-regulation of circRNA_103827 and circRNA_104816 might be potential indicators of compromised follicular micro-environment which could be used to predict IVF prognosis, and improve female infertility management.', 'circRNA_104816');
INSERT INTO `cirrnainfo_relationship` VALUES ('53', 'hsa_circ_0005402', ' ', 'multiple sclerosis', 'PMID: 28651352 ', 'We carried out expression profiling of 13.617 circular RNAs in peripheral blood leucocytes from multiple sclerosis patients and healthy controls finding 406 differentially expressed (P-value<0.05, Fold change>1.5) and demonstrate after validation that, circ_0005402 and circ_0035560 are underexpressed in multiple sclerosis patients and could be used as biomarkers of the disease.', ' ');
INSERT INTO `cirrnainfo_relationship` VALUES ('54', 'hsa_circ_0035560', '', 'multiple sclerosis', 'PMID: 28651352 ', 'We carried out expression profiling of 13.617 circular RNAs in peripheral blood leucocytes from multiple sclerosis patients and healthy controls finding 406 differentially expressed (P-value<0.05, Fold change>1.5) and demonstrate after validation that, circ_0005402 and circ_0035560 are underexpressed in multiple sclerosis patients and could be used as biomarkers of the disease.', 'hsa_circ_0035560');
INSERT INTO `cirrnainfo_relationship` VALUES ('55', 'hsa_circ_0006633', '', 'gastric cancer', 'PMID: 28656881', 'Hsa_circ_0006633 levels at multiple stages of gastric tumorigenesis were then explored, and its relationships with clinicopathological features were analyzed as well. We found that the expression levels of hsa_circ_0006633 in four gastric cancer cell lines, HGC-27, SGC-7901, MGC-803, and AGS, were downregulated than those in normal gastric mucosal epithelial cell line GES-1. Then, we further detected that it was downregulated in 79.2% (76/96) gastric cancer tissues compared with the adjacent non-tumorous tissues. The lower expression of hsa_circ_0006633 was associated with cancer distal metastasis ( p=0.037) and tissue carcinoembryonic antigen level ( p=0.041). In addition, hsa_circ_0006633 expression was significantly decreased in gastritis and dysplasia tissues comparing with the healthy control. Moreover, plasma hsa_circ_0006633 levels were significantly increased in gastric cancer compared with healthy control. Our data imply that hsa_circ_0006633 may play an important role in gastric carcinogenesis and is also a potential biomarker for screening gastric cancer.', 'hsa_circ_0006633');
INSERT INTO `cirrnainfo_relationship` VALUES ('56', '', 'circ-Amotl1', 'wound repair', 'PMID: 28676341', 'In the presence of circ-Amotl1, Stat3 interacted with Dnmt3a promoter with increased affinity, facilitating Dnmt3a transcription. Ectopic application of circ-Amotl1 accelerating wound repair may shed light on skin wound healing clinically.', 'circ-Amotl1');
INSERT INTO `cirrnainfo_relationship` VALUES ('57', '', 'ANRIL', 'coronary atherosclerosis', 'PMID: 28683453', 'Our study provides evidence that reduced cANRIL expression could prevent coronary AS by reducing vascular EC apoptosis and inflammatory factor expression.', 'ANRIL');
INSERT INTO `cirrnainfo_relationship` VALUES ('58', 'hsa_circ_0013958', '', 'lung adenocarcinoma.', 'PMID: 28685964 ', 'The results showed that hsa_circ_0013958 promoted cell proliferation and invasion and inhibited cell apoptosis in LAC. Moreover, hsa_circ_0013958 was identified as a sponge of miR-134, and thus it up-regulated oncogenic cyclin D1, which plays a pivotal role in the development of non-small cell lung cancer. In conclusion, our results suggested that hsa_circ_0013958 could be used as a potential non-invasive biomarker for the early detection and screening of LAC.', 'hsa_circ_0013958');
INSERT INTO `cirrnainfo_relationship` VALUES ('59', '', 'MYLK', 'bladder cancer', 'PMID: 28687357', 'Taken together, this study demonstrated for the first time that circRNA-MYLK might function as competing endogenous RNA (ceRNA) for miR-29a, which could contribute to EMT and the development of BC through activating VEGFA/VEGFR2 and downstream Ras/ERK signaling pathway. Our data suggest that circRNA-MYLK would be a promising target for BC diagnosis and therapy.', 'MYLK');
INSERT INTO `cirrnainfo_relationship` VALUES ('60', '', 'cir-GLI2', 'osteosarcoma', 'PMID: 28695772', 'Furthermore, functional experiments validated that cir-GLI2 exerted the tumor-promoting effects on osteosarcoma cells via negatively targeting miR-125b-5p. In conclusion, our study demonstrated that cir-GLI2 acts as an oncogenic circular RNA in osteosarcoma genesis, providing a novel diagnostic and therapeutic target for osteosarcoma.', 'cir-GLI2');
INSERT INTO `cirrnainfo_relationship` VALUES ('61', 'Hsa_circ_0020397', '', 'Colorectal Cancer', 'PMID: 28707774', 'Hsa_circ_0020397 antagonized miR-138 suppression of cell growth. When TERT or PD-L1 expression was suppressed with siRNAs, the above functions of hsa_circ_0020397 were attenuated, suggesting that hsa_circ_0020397 can regulate CRC cell viability, apoptosis and invasion by promoting the expression of miR-138 target genes. These findings support the role of circRNA in CRC pathogenesis.', 'Hsa_circ_0020397');
INSERT INTO `cirrnainfo_relationship` VALUES ('62', '', 'circRNA-100338', 'hepatocellular carcinoma', 'PMID: 28710406 ', 'Overall, the differential expression of multiple circRNAs in HCC tissues and their clinical significance in hepatitis B-related HCC patients as revealed by our study suggests that circRNA_100338 is a potentially valuable biomarker for HCC diagnosis and target for HCC therapeutics.', 'circRNA-100338');
INSERT INTO `cirrnainfo_relationship` VALUES ('63', '', 'circRNA_021412', 'Hepatic Steatosis', 'PMID: 28717649', 'CircRNA-miRNA-mRNA network further identified the signaling cascade of circRNA_021412/miR-1972/LPIN1, which was characterized by decreased level of circRNA_021412 and miR-1972-based inhibition of LPIN1. LPIN1-induced downregulation of long chain acyl-CoA synthetases (ACSLs) expression finally resulted in the hepatosteatosis. These findings identify circRNAs to be important regulators of hepatic steatosis. Transcription-dependent modulation of metabolic pathways may underlie their effects, partially by the circRNA_021412/miR-1972/LPIN1 signaling.', 'circRNA_021412');
INSERT INTO `cirrnainfo_relationship` VALUES ('64', '', 'circZFR', 'liver cancer', 'PMID: 28727484 ', 'Quantitative real-time PCR was performed to verify the top-five circRNAs. ROC analysis showed circZFR, circFUT8, circIPO11 could significantly distinguish the cancer samples, with an AUC of 0.7069, 0.7575, and 0.7103, respectively. Our results suggest the circRNA-miRNA-mRNA network may help us further understand the molecular mechanisms of tumor progression in liver cancer, and reveal novel biomarkers and therapeutic targets.', 'circZFR');
INSERT INTO `cirrnainfo_relationship` VALUES ('65', '', 'circFUT8', 'liver cancer', 'PMID: 28727484 ', 'Quantitative real-time PCR was performed to verify the top-five circRNAs. ROC analysis showed circZFR, circFUT8, circIPO11 could significantly distinguish the cancer samples, with an AUC of 0.7069, 0.7575, and 0.7103, respectively. Our results suggest the circRNA-miRNA-mRNA network may help us further understand the molecular mechanisms of tumor progression in liver cancer, and reveal novel biomarkers and therapeutic targets.', 'circFUT8');
INSERT INTO `cirrnainfo_relationship` VALUES ('66', '', 'circIPO11', 'liver cancer', 'PMID: 28727484 ', 'Quantitative real-time PCR was performed to verify the top-five circRNAs. ROC analysis showed circZFR, circFUT8, circIPO11 could significantly distinguish the cancer samples, with an AUC of 0.7069, 0.7575, and 0.7103, respectively. Our results suggest the circRNA-miRNA-mRNA network may help us further understand the molecular mechanisms of tumor progression in liver cancer, and reveal novel biomarkers and therapeutic targets.', 'circIPO11');
INSERT INTO `cirrnainfo_relationship` VALUES ('67', '', 'circHIPK3', 'hepatocellular carcinoma', 'PMID: 28738961 ', 'The circular RNA circHIPK3 is abundantly expressed in hepatocellular carcinoma (HCC) and promotes tumourgenesis.', 'circHIPK3');
INSERT INTO `cirrnainfo_relationship` VALUES ('68', '', 'circHIPK3', 'non-small cell lung cancer', 'PMID: 28738961 ', 'CircHIPK3 could promote cell proliferation by a circHIPK3/miR-379 pathway in NCI-H1299 and NCI-H2170 cells and might be a potential tumor biomarker for NSCLC..', 'circHIPK3');
INSERT INTO `cirrnainfo_relationship` VALUES ('69', '', 'circDENND4C', 'breast cancer', 'PMID: 28739726', 'In breast cancer cells, circDENND4C was increased under hypoxic conditions and decreased after knocking-down HIF1. In addition, knocking-down circDENND4C inhibited proliferation of breast cancer cells in a hypoxic environment. Finally, tumors with a large size had higher circDENND4C expression levels than those of small size.', 'circDENND4C');
INSERT INTO `cirrnainfo_relationship` VALUES ('70', '', 'circ-ABCB10', 'breast cancer', 'PMID: 28744405 ', 'Overall, results identified a new functional circ-ABCB10 in breast cancer tumorigenesis, and reveal the important regulatory role of circ-ABCB10 through sponging miR-1271, providing a novel insight for breast cancer pathogenesis.', 'circ-ABCB10');
INSERT INTO `cirrnainfo_relationship` VALUES ('71', '', 'circ-104916', 'gastric cancer', 'PMID: 28761361 ', 'Western blot showed that circ-104916 overexpression upregulated E-cadherin and downregulated N-cadherin, Vimentin and Slug, indicating that circ-104916 was involved in the epithelial-mesenchymal transition process. Our results revealed that circ-104916 might be a novel potential tumor suppressor and biomarker of GC.', 'circ-104916');
INSERT INTO `cirrnainfo_relationship` VALUES ('72', '', 'circSMARCA5', 'prostate cancer', 'PMID: 28765045 ', 'Functional experiments showed circ-SMARCA5 acted as an oncogene in prostate cancer by promoting cell cycle and inhibiting cell apoptosis. We thought this study provided useful information for exploring circRNAs as potential therapeutic and prognostic targets for prostate cancer.', 'circSMARCA5');
INSERT INTO `cirrnainfo_relationship` VALUES ('73', 'hsa_circ_0071410', '', 'Radiation-induced liver fibrosis', 'PMID: 28774651', 'Furthermore, inhibition of hsa_circ_0071410 increased the expression of miR-9-5p, resulting in the attenuation of irradiation induced HSC activation. In summary, this study revealed the expression profile and potential function of differentially expressed circRNAs in irradiated HSC, which provides novel clues for RILF study.', 'hsa_circ_0071410');
INSERT INTO `cirrnainfo_relationship` VALUES ('74', '', 'Hsa-circRNA11783-2', 'type 2 diabetes mellitus', 'PMID: 28777011 ', 'The crude odds ratios and adjusted odds ratios of hsa-circRNA11783-2 showed significant differences in both the coronary artery disease group and type 2 diabetes mellitus group. We then verified hsa-circRNA11783-2 in the third cohort, and it remained closely related to both coronary artery disease and type 2 diabetes mellitus.', 'Hsa-circRNA11783-2');
INSERT INTO `cirrnainfo_relationship` VALUES ('75', '', 'Hsa-circRNA11783-2', 'coronary artery disease', 'PMID: 28777011 ', 'The crude odds ratios and adjusted odds ratios of hsa-circRNA11783-2 showed significant differences in both the coronary artery disease group and type 2 diabetes mellitus group. We then verified hsa-circRNA11783-2 in the third cohort, and it remained closely related to both coronary artery disease and type 2 diabetes mellitus.', 'Hsa-circRNA11783-2');
INSERT INTO `cirrnainfo_relationship` VALUES ('76', '', 'circR-284', 'Carotid Plaque Rupture', 'PMID: 28779016 ', 'Serum circR-284:miR-221 has potential as a diagnostic biomarker of carotid plaque rupture and stroke. Moreover, we demonstrate the use of functionally related pairs of circulating noncoding RNAs as biomarkers in cardiovascular disease.', 'circR-284');
INSERT INTO `cirrnainfo_relationship` VALUES ('77', '', 'circR-284', 'stroke', 'PMID: 28779016 ', 'Serum circR-284:miR-221 has potential as a diagnostic biomarker of carotid plaque rupture and stroke. Moreover, we demonstrate the use of functionally related pairs of circulating noncoding RNAs as biomarkers in cardiovascular disease.', 'circR-284');
INSERT INTO `cirrnainfo_relationship` VALUES ('78', 'hsa_circ_0045714', '', 'osteoarthritis', 'PMID: 28795385 ', 'Therefore, hsa_circ_0045714 can regulate extracellular matrix synthesis as well as proliferation and apoptosis of chondrocytes by promoting the expression of miR-193b target gene IGF1R. The findings will provide new proofs for studies on the applications of circRNA in OA and other orthopedic diseases.', 'hsa_circ_0045714');
INSERT INTO `cirrnainfo_relationship` VALUES ('79', 'hsa_circ_0006528', '', 'breast cancer', 'PMID: 28803498 ', 'The bioinformatics analysis revealed several target genes related to cancer-related signaling pathways. Additionally, we discovered a regulatory role of the circ_0006528-miR-7-5p-Raf1 axis in ADM-resistant breast cancer.', 'hsa_circ_0006528');
INSERT INTO `cirrnainfo_relationship` VALUES ('80', '', 'cZNF609', 'Vascular dysfunction', 'PMID: 28824721 ', 'cZNF609 acted as an endogenous miR-615-5p sponge to sequester and inhibit miR-615-5p activity, which led to increased MEF2A expression. MEF2A overexpression could rescue cZNF609 silencing-mediated effects on endothelial cell migration, tube formation, and apoptosis. Moreover, dysregulated cZNF609 expression was detected in the clinical samples of the patients with diabetes, hypertension, and coronary artery disease. Intervention of cZNF609 expression is promising therapy for vascular dysfunction.', 'cZNF609');
INSERT INTO `cirrnainfo_relationship` VALUES ('81', 'hsa_circ_0005836', '', 'pulmonary tuberculosis', 'PMID: 28846924 ', 'We found for the first time that hsa_circ_0005836 and hsa_circ_0009128 were significantly down-regulated in the PBMCs of APTB compared with HC. Our findings indicate hsa_circ_0005836 might serve as a novel potential biomarker for TB infection.', 'hsa_circ_0005836');
INSERT INTO `cirrnainfo_relationship` VALUES ('82', '', 'circHIPK3 ', 'Diabetes', 'PMID: 28860123 ', 'The circular RNA circHIPK3 plays a role in diabetic retinopathy by blocking miR-30a function, leading to increased endothelial proliferation and vascular dysfunction. These data suggest that circular RNA is a potential target to control diabetic proliferative retinopathy.', 'circHIPK3 ');
INSERT INTO `cirrnainfo_relationship` VALUES ('83', '', 'LARP4', 'gastric cancer', 'PMID: 28893265', 'Increased miR-424 expression or decreased LATS1 expression was associated with pathological stage and unfavorable prognosis of GC patients. Ectopic expression of miR-424 promoted proliferation and invasion of GC cells by targeting LATS1 gene. Furthermore, circLARP4 was mainly localized in the cytoplasm and inhibited biological behaviors of GC cells by sponging miR-424. The expression of circLARP4 was downregulated in GC tissues and represented an independent prognostic factor for overall survival of GC patients.', 'LARP4');
INSERT INTO `cirrnainfo_relationship` VALUES ('84', '', 'Circ-FBXW7', 'brain cancer', 'PMID: 28903484', 'Circ-FBXW7 is abundantly expressed in the normal human brain (reads per kilobase per million mapped reads [RPKM] = 9.31). The spanning junction open reading frame in circ-FBXW7 driven by internal ribosome entry site encodes a novel 21-kDa protein, which we termed FBXW7-185aa. Upregulation of FBXW7-185aa in cancer cells inhibited proliferation and cell cycle acceleration, while knockdown of FBXW7-185aa promoted malignant phenotypes invitro and invivo. FBXW7-185aa reduced the half-life of c-Myc by antagonizing USP28-induced c-Myc stabilization. Moreover, circ-FBXW7 and FBXW7-185aa levels were reduced in glioblastoma clinical samples compared with their paired tumor-adjacent tissues (P < .001). Circ-FBXW7 expression positively associated with glioblastoma patient overall survival (P = .03).', 'Circ-FBXW7');
INSERT INTO `cirrnainfo_relationship` VALUES ('85', 'hsa_circ_0001982', '', 'Breast Cancer', 'PMID: 28933584 ', 'Loss-of-function and rescue experiments revealed that hsa_circ_0001982 knockdown suppressed breast cancer cell proliferation and invasion and induced apoptosis by targeting miR-143. In summary, our study preliminarily investigates the circRNA expression in breast cancer tissue and explores the role of competing endogenous RNA (ceRNA) mechanism in the progression, providing a novel insight for breast cancer tumorigenesis.', 'hsa_circ_0001982');
INSERT INTO `cirrnainfo_relationship` VALUES ('86', 'hsa_circ_0004018 ', '', 'hepatocellular carcinoma', 'PMID: 28938566 ', 'Our data further showed that lower expression of hsa_circ_0004018 was correlated with serum alpha-fetoprotein (AFP) level, tumor diameters, differentiation, Barcelona Clinic Liver Cancer stage and Tumor-node-metastasis stage. More importantly, we detected liver tissues from chronic hepatitis, cirrhosis and HCC patients; and found that hsa_circ_0004018 harbored HCC-stage-specific expression features in diverse chronic liver diseases (P<0.001). The area under receiver operating characteristic curve was up to 0.848 (95% CI=0.803-0.894, P<0.001). The sensitivity and specificity were 0.716 and 0.815, respectively. Finally, hsa_circ_0004018 might be involved in cancer-related pathways via interactions with miRNAs.', 'hsa_circ_0004018 ');
INSERT INTO `cirrnainfo_relationship` VALUES ('87', 'hsa_circ_0000181', '', 'gastric cancer', 'PMID: 28940688', 'Hsa_circ_0000181 levels in gastric cancer tissues and plasma from gastric cancer patients were significantly decreased than those in paired adjacent non-tumorous tissues (P<.001) and healthy people (P<.001), respectively. Furthermore, hsa_circ_0000181 expression in gastric cancer tissues was significantly correlated with tumor diameter (P=.027), lymphatic metastasis (P=.044), distal metastasis (P=.023), and carbohydrate antigen 19-9 (P=.031). Its decreased levels in patients^ plasma were significantly associated with differentiation (P=.038) and carcinoembryonic antigen (P=.037). The areas under ROC curve were 0.756. The specificity of tissue hsa_circ_0000181 and sensitivity of plasma hsa_circ_0000181 were 85.2% and 99.0%, respectively.', 'hsa_circ_0000181');
INSERT INTO `cirrnainfo_relationship` VALUES ('88', 'hsa_circ_0003575', '', 'atherosclerosis', 'PMID: 28946214', 'In summary, circRNA microarray analysis reveals the expression profiles of HUVECs and verifies the role of hsa_circ_0003575 on HUVECs, providing a therapeutic strategy for vascular endothelial cell injury of atherosclerosis.', 'hsa_circ_0003575');
INSERT INTO `cirrnainfo_relationship` VALUES ('89', '', 'mcircRasGEF1B', 'immune response', 'PMID: 28947785', 'These findings expand the set of functionally characterized circRNAs and support the regulatory role of mcircRasGEF1B in immune response during macrophage activation and protection against microbial infections', 'mcircRasGEF1B');
INSERT INTO `cirrnainfo_relationship` VALUES ('90', 'hsa_circ_0043256', '', 'lung cancer', 'PMID: 28958934 ', 'Interestingly, hsa_circ_0043256 knock-down could weaken Wnt/-catenin pathway inhibition induced by CA. Finally, we discovered that CA induced apoptosis and meanwhile upregulated hsa_circ_0043256 expression invivo. Immunohistochemical analysis revealed that ITCH expression was positively association with hsa_circ_0043256 levels. Above all, we characterized a new mechanism mediated by hsa_circ_0043256/miR-1252/ITCH axis in CA function against NSCLC, providing a novel insight into lung cancer therapy.', 'hsa_circ_0043256');
INSERT INTO `cirrnainfo_relationship` VALUES ('91', 'hsa_circ_0000745', '', 'gastric cancer', 'PMID: 28974900 ', 'Hsa_circ_0000745 was down-regulated in GC tissues vs non-tumorous tissues (P < 0.001) and in plasma samples from patients with GC vs healthy controls (P < 0.001). The expression level of hsa_circ_0000745 in GC tissues correlated with tumor differentiation, while the expression level in plasma correlated with tumor-node-metastasis stage. The area under the ROC curve (AUC) of hsa_circ_0000745 in plasma was 0.683, suggesting good diagnostic value. Plasma hsa_circ_0000745 level combined with carcinoembryogenic antigen (CEA) level increased the AUC to 0.775.', 'hsa_circ_0000745');
INSERT INTO `cirrnainfo_relationship` VALUES ('92', '', 'circUBAP2', 'osteosarcoma ', 'PMID: 28977896', 'Mechanistically, circUBAP2 was found to inhibit the expression of microRNA-143 (miR-143), thus enhancing the expression and function of anti-apoptotic Bcl-2, which is a direct target of miR-143. Together, our results suggest the roles of circUBAP2 in osteosarcoma development and implicate its potential in prognosis prediction and cancer therapy.', 'circUBAP2');
INSERT INTO `cirrnainfo_relationship` VALUES ('93', '', 'has-circRNA7690-15', 'gastric cancer', 'PMID: 28980874 ', 'Additionally, hsa_circ_0047905 and hsa_circ_0138960 positively correlated with their parental gene mRNA. Knockdown of hsa_circ_0047905, hsa_circ_0138960 and has-circRNA7690-15 in GC cells, resulted in downregulation of parental gene expression. Functional assays suggested that inhibition of these three circular RNAs suppresses GC cell proliferation and invasion in vitro. Those findings suggest that hsa_circ_0047905, hsa_circ_0138960 and has-circRNA7690-15 might act as tumor promoters in the pathogenesis of gastric cancer.', 'has-circRNA7690-15');
INSERT INTO `cirrnainfo_relationship` VALUES ('94', 'hsa_circ_0047905', '', 'gastric cancer', 'PMID: 28980874 ', 'Additionally, hsa_circ_0047905 and hsa_circ_0138960 positively correlated with their parental gene mRNA. Knockdown of hsa_circ_0047905, hsa_circ_0138960 and has-circRNA7690-15 in GC cells, resulted in downregulation of parental gene expression. Functional assays suggested that inhibition of these three circular RNAs suppresses GC cell proliferation and invasion in vitro. Those findings suggest that hsa_circ_0047905, hsa_circ_0138960 and has-circRNA7690-15 might act as tumor promoters in the pathogenesis of gastric cancer.', 'hsa_circ_0047905');
INSERT INTO `cirrnainfo_relationship` VALUES ('95', 'hsa_circ_0138960', '', 'gastric cancer', 'PMID: 28980874 ', 'Additionally, hsa_circ_0047905 and hsa_circ_0138960 positively correlated with their parental gene mRNA. Knockdown of hsa_circ_0047905, hsa_circ_0138960 and has-circRNA7690-15 in GC cells, resulted in downregulation of parental gene expression. Functional assays suggested that inhibition of these three circular RNAs suppresses GC cell proliferation and invasion in vitro. Those findings suggest that hsa_circ_0047905, hsa_circ_0138960 and has-circRNA7690-15 might act as tumor promoters in the pathogenesis of gastric cancer.', 'hsa_circ_0138960');
INSERT INTO `cirrnainfo_relationship` VALUES ('96', '', 'hsa_circRNA_101407', 'Rheumatoid arthritis', 'PMID: 28983619 ', 'RTqPCR validation demonstrated that the expression levels of hsa_circRNA_104194, hsa_circRNA_104593, hsa_circRNA_103334, hsa_circRNA_101407 and hsa_circRNA_102594 were consistent with the results from the microarray analysis. The current study presented differentially expressed circRNAs and their corresponding microRNA binding sites in RA. circRNAs may exhibit a role in the regulation of expression of symbol genes that influence the occurrence and development of RA.', 'hsa_circRNA_101407');
INSERT INTO `cirrnainfo_relationship` VALUES ('97', '', 'hsa_circRNA_102594', 'Rheumatoid arthritis', 'PMID: 28983619 ', 'RTqPCR validation demonstrated that the expression levels of hsa_circRNA_104194, hsa_circRNA_104593, hsa_circRNA_103334, hsa_circRNA_101407 and hsa_circRNA_102594 were consistent with the results from the microarray analysis. The current study presented differentially expressed circRNAs and their corresponding microRNA binding sites in RA. circRNAs may exhibit a role in the regulation of expression of symbol genes that influence the occurrence and development of RA.', 'hsa_circRNA_102594');
INSERT INTO `cirrnainfo_relationship` VALUES ('98', '', 'hsa_circRNA_103334', 'Rheumatoid arthritis', 'PMID: 28983619 ', 'RTqPCR validation demonstrated that the expression levels of hsa_circRNA_104194, hsa_circRNA_104593, hsa_circRNA_103334, hsa_circRNA_101407 and hsa_circRNA_102594 were consistent with the results from the microarray analysis. The current study presented differentially expressed circRNAs and their corresponding microRNA binding sites in RA. circRNAs may exhibit a role in the regulation of expression of symbol genes that influence the occurrence and development of RA.', 'hsa_circRNA_103334');
INSERT INTO `cirrnainfo_relationship` VALUES ('99', '', 'hsa_circRNA_104194', 'Rheumatoid arthritis', 'PMID: 28983619 ', 'RTqPCR validation demonstrated that the expression levels of hsa_circRNA_104194, hsa_circRNA_104593, hsa_circRNA_103334, hsa_circRNA_101407 and hsa_circRNA_102594 were consistent with the results from the microarray analysis. The current study presented differentially expressed circRNAs and their corresponding microRNA binding sites in RA. circRNAs may exhibit a role in the regulation of expression of symbol genes that influence the occurrence and development of RA.', 'hsa_circRNA_104194');
INSERT INTO `cirrnainfo_relationship` VALUES ('100', '', 'hsa_circRNA_104593', 'Rheumatoid arthritis', 'PMID: 28983619 ', 'RTqPCR validation demonstrated that the expression levels of hsa_circRNA_104194, hsa_circRNA_104593, hsa_circRNA_103334, hsa_circRNA_101407 and hsa_circRNA_102594 were consistent with the results from the microarray analysis. The current study presented differentially expressed circRNAs and their corresponding microRNA binding sites in RA. circRNAs may exhibit a role in the regulation of expression of symbol genes that influence the occurrence and development of RA.', 'hsa_circRNA_104593');
INSERT INTO `cirrnainfo_relationship` VALUES ('101', '', 'circRNA_0046367', 'Hepatic steatosis', 'PMID: 29018509 ', 'These findings indicate a circRNA_0046367/miR-34a/PPAR regulatory system underlying hepatic steatosis. Normalized expression of circRNA_0046367 may ameliorate the lipoxidative stress on the basis of steatosis attenuation. circRNA_0046367, therefore, is suggested to be potential approach to the therapy of lipid peroxidative damage.', 'circRNA_0046367');
INSERT INTO `cirrnainfo_relationship` VALUES ('102', '', 'hsa_circRNA_086376', 'colorectal cancer with lung metastasis', 'PMID: 29039473', 'In addition, it was demonstrated that hsa_circRNA_105055, hsa_circRNA_086376 and hsa_circRNA_102761 could commonly bind with miR-7 regulating target genes PRKCB, EPHA3, BRCA1 and ABCC1. The findings of the present study may provide a novel perspective on circRNA and lay a foundation for future research of potential roles of circRNA in CRC with lung metastasis.', 'hsa_circRNA_086376');
INSERT INTO `cirrnainfo_relationship` VALUES ('103', '', 'hsa_circRNA_102761', 'colorectal cancer with lung metastasis', 'PMID: 29039473', 'In addition, it was demonstrated that hsa_circRNA_105055, hsa_circRNA_086376 and hsa_circRNA_102761 could commonly bind with miR-7 regulating target genes PRKCB, EPHA3, BRCA1 and ABCC1. The findings of the present study may provide a novel perspective on circRNA and lay a foundation for future research of potential roles of circRNA in CRC with lung metastasis.', 'hsa_circRNA_102761');
INSERT INTO `cirrnainfo_relationship` VALUES ('104', '', 'hsa_circRNA_105055', 'colorectal cancer with lung metastasis', 'PMID: 29039473', 'In addition, it was demonstrated that hsa_circRNA_105055, hsa_circRNA_086376 and hsa_circRNA_102761 could commonly bind with miR-7 regulating target genes PRKCB, EPHA3, BRCA1 and ABCC1. The findings of the present study may provide a novel perspective on circRNA and lay a foundation for future research of potential roles of circRNA in CRC with lung metastasis.', 'hsa_circRNA_105055');
INSERT INTO `cirrnainfo_relationship` VALUES ('105', '', 'circWDR77', 'diabetes mellitus correlated vasculopathy', 'PMID: 29042195 ', 'Further function validation experiments revealed that circWDR77 regulated VSMCs proliferation and migration via targeting miR-124/FGF2. Taken together, present study firstly reveals the circRNAs expression profiles in high glucose induced VSMCs and identifies the role of circWDR77-miR-124-FGF2 regulatory pathway in VSMCs proliferation and migration, which might provide a new theoretical basis for diabetes mellitus correlated vasculopathy.', 'circWDR77');
INSERT INTO `cirrnainfo_relationship` VALUES ('106', 'hsa_circ_0001785', '', 'breast cancer', 'PMID: 29045858 ', 'Our study reveals the aberrant circRNAs expression profiles in breast cancer peripheral blood, and identifies the potential diagnostic value of plasma hsa_circ_0001785, providing a stable biomarkers for the diagnosis and progress of breast cancer.', 'hsa_circ_0001785');
INSERT INTO `cirrnainfo_relationship` VALUES ('107', 'Hsa_circ_0000520', '', 'gastric carcinoma', 'PMID:29103021', 'In this study, hsa_circ_0000520 was first found to be significantly down-regulated in gastric cancer tissues, plasma and gastric cancer cell lines compared with control cases. Clinicopathological features showed that hsa_circ_0000520 level in GC tissues was negatively associated with TNM stage and in GC plasma linked with CEA expression. Finally, a total of 9 miRNAs and 9 candidate mRNA were predicted to have an interaction with hsa_circ_0000520.', 'Hsa_circ_0000520');
INSERT INTO `cirrnainfo_relationship` VALUES ('108', '', 'circRNA0003906', 'colorectal cancer', 'PMID:29123417', 'Our results showed that circRNA0003906 expression level was dramatically downregulated in both CRC tissues and cell lines. Moreover, the downregulation of circRNA0003906 level significantly correlated with lymphatic metastasis and poor differentiation. In addition, the area under the receiver operating characteristic curve of circRNA0003906 for CRC was 0.818 (P<0.001). Taking consideration of all of these results, circRNA0003906 may be potentially involved in the colorectal cancerogenesis and serve as a potential biomarker for the diagnosis and treatment of CRC.', 'circRNA0003906');
INSERT INTO `cirrnainfo_relationship` VALUES ('109', 'Hsa_circ_0001017', '', 'gastric cancer', 'PMID:29098316', 'A total of 343 circRNAs are differentially expressed between gastric cancer patients^ plasma and healthy controls. Hsa_circ_0001017 and hsa_circ_0061276 are downregulated in gastric cancer tissues. The RT-ddPCR is a potent method for simultaneous detection of multiple circRNAs in plasma. Hsa_circ_0001017 and hsa_circ_0061276 are potential biomarkers for gastric cancer.', 'Hsa_circ_0001017');
INSERT INTO `cirrnainfo_relationship` VALUES ('110', 'hsa_circ_0061276', '', 'gastric cancer', 'PMID:29098316', 'A total of 343 circRNAs are differentially expressed between gastric cancer patients^ plasma and healthy controls. Hsa_circ_0001017 and hsa_circ_0061276 are downregulated in gastric cancer tissues. The RT-ddPCR is a potent method for simultaneous detection of multiple circRNAs in plasma. Hsa_circ_0001017 and hsa_circ_0061276 are potential biomarkers for gastric cancer.', 'hsa_circ_0061276');
INSERT INTO `cirrnainfo_relationship` VALUES ('111', '', 'hsa_circRNA_100933', 'infantile hemangioma', 'PMID:29095957', '234 up- and 374 down- regulated circRNAs were identified in IH by microarray. Among them, the expression of two up-regulated circRNAs (hsa_circRNA_100933 and hsa_circRNA_100709) and one down-regulated circRNA (hsa_circRNA_104310) was confirmed by qRT-PCR. In addition, 3,019 miRNA response elements (MREs) of circRNAs were predicted, and two circRNA-miRNA-mRNA networks were constructed, including 100 and 94 target genes of hsa_circRNA_100933 and hsa_circRNA_104310, respectively. GO and pathway analysis showed that both networks participated in angiogenesis and vascular development-related biological processes.', 'hsa_circRNA_100933');
INSERT INTO `cirrnainfo_relationship` VALUES ('112', '', 'hsa_circRNA_104310', 'infantile hemangioma', 'PMID:29095957', '234 up- and 374 down- regulated circRNAs were identified in IH by microarray. Among them, the expression of two up-regulated circRNAs (hsa_circRNA_100933 and hsa_circRNA_100709) and one down-regulated circRNA (hsa_circRNA_104310) was confirmed by qRT-PCR. In addition, 3,019 miRNA response elements (MREs) of circRNAs were predicted, and two circRNA-miRNA-mRNA networks were constructed, including 100 and 94 target genes of hsa_circRNA_100933 and hsa_circRNA_104310, respectively. GO and pathway analysis showed that both networks participated in angiogenesis and vascular development-related biological processes.', 'hsa_circRNA_104310');
INSERT INTO `cirrnainfo_relationship` VALUES ('113', '', 'hsa_circRNA_070616', 'Repeated Implantation Failure', 'PMID:29132137', 'The data from circRNA microarrays clearly revealed that 856 unique circRNAs were significantly altered (p<0.05). The up-regulated expression of hsa_circRNA_070616, hsa_circRNA_103716, hsa_circRNA_104001, hsa_circRNA_104854 and the down-regulated expression of hsa_circRNA_004183, hsa_circRNA_044353, hsa_circRNA_404686 were further validated by qRT-PCR.', 'hsa_circRNA_070616');
INSERT INTO `cirrnainfo_relationship` VALUES ('114', '', 'hsa_circRNA_104854', 'Repeated Implantation Failure', 'PMID:29132137', 'The data from circRNA microarrays clearly revealed that 856 unique circRNAs were significantly altered (p<0.05). The up-regulated expression of hsa_circRNA_070616, hsa_circRNA_103716, hsa_circRNA_104001, hsa_circRNA_104855 and the down-regulated expressio\0', 'hsa_circRNA_104854');
INSERT INTO `cirrnainfo_relationship` VALUES ('115', '', 'hsa_circRNA_104001', 'Repeated Implantation Failure', 'PMID:29132137', 'The data from circRNA microarrays clearly revealed that 856 unique circRNAs were significantly altered (p<0.05). The up-regulated expression of hsa_circRNA_070616, hsa_circRNA_103716, hsa_circRNA_104001, hsa_circRNA_104856 and the down-regulated expressio\0\0\0\0\0', 'hsa_circRNA_104001');
INSERT INTO `cirrnainfo_relationship` VALUES ('116', '', 'hsa_circRNA_103716', 'Repeated Implantation Failure', 'PMID:29132137', 'The data from circRNA microarrays clearly revealed that 856 unique circRNAs were significantly altered (p<0.05). The up-regulated expression of hsa_circRNA_070616, hsa_circRNA_103716, hsa_circRNA_104001, hsa_circRNA_104857 and the down-regulated expressio\0\0\0\0\0', 'hsa_circRNA_103716');
INSERT INTO `cirrnainfo_relationship` VALUES ('117', 'hsa_circ_0003221', 'circPTK2', 'bladder cancer', 'PMID:29125888', 'Quantitative PCR results confirmed that circPTK2 is highly expressed in migrated cells separated by Transwell assay and in metastatic lymph nodes of tumors transplanted in nude mice. In vitro silence of circPTK2 by small interfering RNA inhibited the proliferation and migration of bladder cancer cells. On the contrary, circPTK2 overexpression promoted proliferation and migration. This study showed that circPTK2 promotes proliferation and migration of cells and may be a novel potential biomarker and therapeutic target for bladder cancer diagnosis and therapy.', 'circPTK2');
INSERT INTO `cirrnainfo_relationship` VALUES ('118', '', 'circDLGAP4', 'stroke', 'PMID:29114076', 'Here, we demonstrate a critical role for circular RNA DLGAP4 (circDLGAP4), a novel circular RNA originally identified as a sponge for microRNA-143 (miR-143), in ischemic stroke outcomes. Overexpression of circDLGAP4 significantly attenuated neurological deficits and decreased infarct areas and blood-brain barrier (BBB) damage in the tMCAO (transient middle cerebral artery occlusion) mouse stroke model. To our knowledge, this is the first report describing the efficacy of circular RNA injection in an ischemic stroke model. Our investigation suggests that circDLGAP4 may serve as a novel therapeutic target for acute ischemic injury.', 'circDLGAP4');

-- ----------------------------
-- Table structure for django_admin_log
-- ----------------------------
DROP TABLE IF EXISTS `django_admin_log`;
CREATE TABLE `django_admin_log` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `action_time` datetime(6) NOT NULL,
  `object_id` longtext,
  `object_repr` varchar(200) NOT NULL,
  `action_flag` smallint(5) unsigned NOT NULL,
  `change_message` longtext NOT NULL,
  `content_type_id` int(11) DEFAULT NULL,
  `user_id` int(11) NOT NULL,
  PRIMARY KEY (`id`),
  KEY `django_admin_log_content_type_id_c4bce8eb_fk_django_co` (`content_type_id`),
  KEY `django_admin_log_user_id_c564eba6_fk_auth_user_id` (`user_id`),
  CONSTRAINT `django_admin_log_content_type_id_c4bce8eb_fk_django_co` FOREIGN KEY (`content_type_id`) REFERENCES `django_content_type` (`id`),
  CONSTRAINT `django_admin_log_user_id_c564eba6_fk_auth_user_id` FOREIGN KEY (`user_id`) REFERENCES `auth_user` (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8;

-- ----------------------------
-- Records of django_admin_log
-- ----------------------------

-- ----------------------------
-- Table structure for django_content_type
-- ----------------------------
DROP TABLE IF EXISTS `django_content_type`;
CREATE TABLE `django_content_type` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `app_label` varchar(100) NOT NULL,
  `model` varchar(100) NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `django_content_type_app_label_model_76bd3d3b_uniq` (`app_label`,`model`)
) ENGINE=InnoDB AUTO_INCREMENT=10 DEFAULT CHARSET=utf8;

-- ----------------------------
-- Records of django_content_type
-- ----------------------------
INSERT INTO `django_content_type` VALUES ('4', 'admin', 'logentry');
INSERT INTO `django_content_type` VALUES ('6', 'auth', 'group');
INSERT INTO `django_content_type` VALUES ('5', 'auth', 'permission');
INSERT INTO `django_content_type` VALUES ('7', 'auth', 'user');
INSERT INTO `django_content_type` VALUES ('2', 'cirRNAInfo', 'cirrna');
INSERT INTO `django_content_type` VALUES ('1', 'cirRNAInfo', 'disease');
INSERT INTO `django_content_type` VALUES ('3', 'cirRNAInfo', 'relationship');
INSERT INTO `django_content_type` VALUES ('8', 'contenttypes', 'contenttype');
INSERT INTO `django_content_type` VALUES ('9', 'sessions', 'session');

-- ----------------------------
-- Table structure for django_migrations
-- ----------------------------
DROP TABLE IF EXISTS `django_migrations`;
CREATE TABLE `django_migrations` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `app` varchar(255) NOT NULL,
  `name` varchar(255) NOT NULL,
  `applied` datetime(6) NOT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=19 DEFAULT CHARSET=utf8;

-- ----------------------------
-- Records of django_migrations
-- ----------------------------
INSERT INTO `django_migrations` VALUES ('1', 'contenttypes', '0001_initial', '2018-03-10 04:58:03.709709');
INSERT INTO `django_migrations` VALUES ('2', 'auth', '0001_initial', '2018-03-10 04:58:04.677832');
INSERT INTO `django_migrations` VALUES ('3', 'admin', '0001_initial', '2018-03-10 04:58:04.907423');
INSERT INTO `django_migrations` VALUES ('4', 'admin', '0002_logentry_remove_auto_add', '2018-03-10 04:58:04.919932');
INSERT INTO `django_migrations` VALUES ('5', 'contenttypes', '0002_remove_content_type_name', '2018-03-10 04:58:05.096556');
INSERT INTO `django_migrations` VALUES ('6', 'auth', '0002_alter_permission_name_max_length', '2018-03-10 04:58:05.193620');
INSERT INTO `django_migrations` VALUES ('7', 'auth', '0003_alter_user_email_max_length', '2018-03-10 04:58:05.273422');
INSERT INTO `django_migrations` VALUES ('8', 'auth', '0004_alter_user_username_opts', '2018-03-10 04:58:05.284432');
INSERT INTO `django_migrations` VALUES ('9', 'auth', '0005_alter_user_last_login_null', '2018-03-10 04:58:05.362073');
INSERT INTO `django_migrations` VALUES ('10', 'auth', '0006_require_contenttypes_0002', '2018-03-10 04:58:05.365075');
INSERT INTO `django_migrations` VALUES ('11', 'auth', '0007_alter_validators_add_error_messages', '2018-03-10 04:58:05.375082');
INSERT INTO `django_migrations` VALUES ('12', 'auth', '0008_alter_user_username_max_length', '2018-03-10 04:58:05.465418');
INSERT INTO `django_migrations` VALUES ('13', 'cirRNAInfo', '0001_initial', '2018-03-10 04:58:05.513314');
INSERT INTO `django_migrations` VALUES ('14', 'cirRNAInfo', '0002_cirrna', '2018-03-10 04:58:05.571872');
INSERT INTO `django_migrations` VALUES ('15', 'cirRNAInfo', '0003_relationship', '2018-03-10 04:58:05.617492');
INSERT INTO `django_migrations` VALUES ('16', 'cirRNAInfo', '0004_auto_20171130_2145', '2018-03-10 04:58:05.640057');
INSERT INTO `django_migrations` VALUES ('17', 'cirRNAInfo', '0005_relationship_displaycir_id', '2018-03-10 04:58:05.731243');
INSERT INTO `django_migrations` VALUES ('18', 'sessions', '0001_initial', '2018-03-10 04:58:05.794973');

-- ----------------------------
-- Table structure for django_session
-- ----------------------------
DROP TABLE IF EXISTS `django_session`;
CREATE TABLE `django_session` (
  `session_key` varchar(40) NOT NULL,
  `session_data` longtext NOT NULL,
  `expire_date` datetime(6) NOT NULL,
  PRIMARY KEY (`session_key`),
  KEY `django_session_expire_date_a5c62663` (`expire_date`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8;

-- ----------------------------
-- Records of django_session
-- ----------------------------
